#!/usr/bin/env python
# File created on 27 Sep 2012
from __future__ import division

__author__ = "Yoshiki Vazquez-Baeza"
__copyright__ = "Copyright 2011-2012, Evident"
__credits__ = ["Yoshiki Vazquez-Baeza"]
__license__ = "GPL"
__version__ = "0.9-dev"
__maintainer__ = "Yoshiki Vazquez-Baeza"
__email__ = "yoshiki89@gmail.com"
__status__ = "Development"


from cogent.util.unit_test import TestCase, main
from qiime.util import get_qiime_temp_dir, get_tmp_filename

from numpy import array
from biom.parse import parse_biom_table
from biom.exception import TableException
from evident.evident import make_pcoa_plot
from os.path import abspath, dirname, exists, split

class TopLevelTests(TestCase):
    def setUp(self):
        self.ellipses_input_a = input_ellipses_dict_a
        self.mapping_file_tuple_a = input_mapping_file_tuple_a
        self.pcoalabels_a = input_pcoalabels_a

        self.ellipses_input_b = input_ellipses_dict_b
        self.mapping_file_tuple_b = input_mapping_file_tuple_b
        self.pcoalabels_b = input_pcoalabels_b

        self.list_of_study_names = ['hmp-v13', 'hmp-v35', 'keyboard',\
                                    'turnbaugh-twins', 'wholebody']
        self.list_of_file_suffixes = ['.biom', '_map.txt', '_selectors.txt',\
                                        '_unweighted_unifrac_pc.txt',
                                        '_alpha_stddev.html',
                                        '_alpha_stderr.html']

        self.list_of_resources = ['gg_97_otus_4feb2011.tre', 'studies.txt']

        # assuming this file lives in the test dir along with the data dir
        evident_dir, test_dir = split(dirname(abspath(__file__)))
        self.data_path = ''.join([evident_dir,'/data/'])

    def test_make_pcoa_plot(self):
        """checks that the function to create WebGL strings is working"""
        # known working cases
        output_webgl_string = make_pcoa_plot(self.ellipses_input_a,\
            self.mapping_file_tuple_a, self.pcoalabels_a)
        self.assertEquals(output_webgl_string, expected_output_webgl_string_a)

        # demo case for fierer keyboard study
        output_webgl_string = make_pcoa_plot(self.ellipses_input_b,\
            self.mapping_file_tuple_b, self.pcoalabels_b)
        self.assertEquals(output_webgl_string, expected_output_webgl_string_b)
		
        # broken case
        with self.assertRaises(IndexError):
            output_webgl_string = make_pcoa_plot({},[],[])

    def test_demo_files_are_in_place(self):
        """checks that the files e-vident needs to run the demos are in place"""
        # test the files that are supposed to exist for a demo are there
        for study_name in self.list_of_study_names:
            for study_suffix in self.list_of_file_suffixes:
                testing_file = ''.join([self.data_path, study_name, study_suffix])

                self.assertTrue(exists(testing_file), 'Could not find %s' % testing_file)

    def test_resources_are_in_place(self):
        """checks that the files e-vident needs to function are in place"""
        for resource_name in self.list_of_resources:
            testing_file = ''.join([self.data_path,resource_name])
            self.assertTrue(exists(testing_file), 'Could not find %s' % testing_file)


input_ellipses_dict_a= {'M2Thmr217.141087': {'axes_radii': [0.0, 0.0, 0.0], 'center': [-0.047183222753100003, -0.0265524618955, 0.242405807261]}, 'M2Nkey217.141052': {'axes_radii': [0.0, 0.0, 0.0], 'center': [-0.13075544576799999, -0.119444606523, 0.022675239445099998]}, 'M2Midl217.141009': {'axes_radii': [0.0, 0.0, 0.0], 'center': [0.35119680235299999, 0.0059234484597300002, -0.059677044456999999]}, 'M2Mkey217.141014': {'axes_radii': [0.0, 0.0, 0.0], 'center': [-0.20525484646299999, -0.044230407698800003, -0.18072459397400001]}, 'M2Akey217.141030': {'axes_radii': [0.0, 0.0, 0.0], 'center': [0.090078990788499999, -0.116932498737, -0.0213770416347]}, 'M2Rinr217.141027': {'axes_radii': [0.0, 0.0, 0.0], 'center': [-0.0580822781577, 0.301236526394, -0.00330236664045]}}
input_mapping_file_tuple_a = ([['M2Thmr217.141087', 'AATCAGTCTCGT', 'M2Thmr217_V2', 'human skin metagenome', '539655', 'UBERON:skin of finger', 'ENVO:human-associated habitat', 'M2Thmr217', 'UBERON:skin', 'M2Thmr217_V2', 'ENVO:sebum', 'ENVO:human-associated habitat', 'UBERON:sebum', 'finger_tip'], ['M2Akey217.141030', 'ACATGATCGTTC', 'M2Akey217_V2', 'computer keyboard metagenome', '256318', 'NA', 'ENVO:anthropogenic habitat', 'M2Akey217', 'NA', 'M2Akey217_V2', 'ENVO:surface', 'ENVO:surface', 'NA', 'Akey'], ['M2Rinr217.141027', 'ACACATGTCTAC', 'M2Rinr217_V2', 'human skin metagenome', '539655', 'UBERON:skin of finger', 'ENVO:human-associated habitat', 'M2Rinr217', 'UBERON:skin', 'M2Rinr217_V2', 'ENVO:sebum', 'ENVO:human-associated habitat', 'UBERON:sebum', 'finger_tip'], ['M2Mkey217.141014', 'ACGCTATCTGGA', 'M2Mkey217_V2', 'computer keyboard metagenome', '256318', 'NA', 'ENVO:anthropogenic habitat', 'M2Mkey217', 'NA', 'M2Mkey217_V2', 'ENVO:surface', 'ENVO:surface', 'NA', 'Mkey'], ['M2Nkey217.141052', 'ACGCGCAGATAC', 'M2Nkey217_V2', 'computer keyboard metagenome', '256318', 'NA', 'ENVO:anthropogenic habitat', 'M2Nkey217', 'NA', 'M2Nkey217_V2', 'ENVO:surface', 'ENVO:surface', 'NA', 'Nkey'], ['M2Midl217.141009', 'AACTGTGCGTAC', 'M2Midl217_V2', 'human skin metagenome', '539655', 'UBERON:skin of finger', 'ENVO:human-associated habitat', 'M2Midl217', 'UBERON:skin', 'M2Midl217_V2', 'ENVO:sebum', 'ENVO:human-associated habitat', 'UBERON:sebum', 'finger_tip']], ['SampleID', 'BarcodeSequence', 'BARCODE_READ_GROUP_TAG', 'COMMON_NAME', 'TAXON_ID', 'BODY_SITE', 'ENV_BIOME', 'ANONYMIZED_NAME', 'BODY_HABITAT', 'POOL_MEMBER_NAME', 'ENV_MATTER', 'ENV_FEATURE', 'BODY_PRODUCT', 'Description'])
input_pcoalabels_a = [23.5,8.2,2.4]
expected_output_webgl_string_a = """var headers = ["SampleID","BarcodeSequence","BARCODE_READ_GROUP_TAG","COMMON_NAME","TAXON_ID","BODY_SITE","ENV_BIOME","ANONYMIZED_NAME","BODY_HABITAT","POOL_MEMBER_NAME","ENV_MATTER","ENV_FEATURE","BODY_PRODUCT","Description"]
var mapping = {"M2Thmr217.141087": ["M2Thmr217.141087","AATCAGTCTCGT","M2Thmr217_V2","human skin metagenome","539655","UBERON:skin of finger","ENVO:human-associated habitat","M2Thmr217","UBERON:skin","M2Thmr217_V2","ENVO:sebum","ENVO:human-associated habitat","UBERON:sebum","finger_tip"],"M2Akey217.141030": ["M2Akey217.141030","ACATGATCGTTC","M2Akey217_V2","computer keyboard metagenome","256318","NA","ENVO:anthropogenic habitat","M2Akey217","NA","M2Akey217_V2","ENVO:surface","ENVO:surface","NA","Akey"],"M2Rinr217.141027": ["M2Rinr217.141027","ACACATGTCTAC","M2Rinr217_V2","human skin metagenome","539655","UBERON:skin of finger","ENVO:human-associated habitat","M2Rinr217","UBERON:skin","M2Rinr217_V2","ENVO:sebum","ENVO:human-associated habitat","UBERON:sebum","finger_tip"],"M2Mkey217.141014": ["M2Mkey217.141014","ACGCTATCTGGA","M2Mkey217_V2","computer keyboard metagenome","256318","NA","ENVO:anthropogenic habitat","M2Mkey217","NA","M2Mkey217_V2","ENVO:surface","ENVO:surface","NA","Mkey"],"M2Nkey217.141052": ["M2Nkey217.141052","ACGCGCAGATAC","M2Nkey217_V2","computer keyboard metagenome","256318","NA","ENVO:anthropogenic habitat","M2Nkey217","NA","M2Nkey217_V2","ENVO:surface","ENVO:surface","NA","Nkey"],"M2Midl217.141009": ["M2Midl217.141009","AACTGTGCGTAC","M2Midl217_V2","human skin metagenome","539655","UBERON:skin of finger","ENVO:human-associated habitat","M2Midl217","UBERON:skin","M2Midl217_V2","ENVO:sebum","ENVO:human-associated habitat","UBERON:sebum","finger_tip"]}
var ellipses = new Array()
var points = new Array()
ellipses['M2Thmr217.141087'] = { 'name': 'M2Thmr217.141087', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.047183, 'y': -0.026552, 'z': 0.242406 }
points['M2Thmr217.141087'] = { 'name': 'M2Thmr217.141087', 'color': 0, 'x': -0.047183, 'y': -0.026552, 'z': 0.242406 }
ellipses['M2Nkey217.141052'] = { 'name': 'M2Nkey217.141052', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.130755, 'y': -0.119445, 'z': 0.022675 }
points['M2Nkey217.141052'] = { 'name': 'M2Nkey217.141052', 'color': 0, 'x': -0.130755, 'y': -0.119445, 'z': 0.022675 }
ellipses['M2Midl217.141009'] = { 'name': 'M2Midl217.141009', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.351197, 'y': 0.005923, 'z': -0.059677 }
points['M2Midl217.141009'] = { 'name': 'M2Midl217.141009', 'color': 0, 'x': 0.351197, 'y': 0.005923, 'z': -0.059677 }
ellipses['M2Mkey217.141014'] = { 'name': 'M2Mkey217.141014', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.205255, 'y': -0.044230, 'z': -0.180725 }
points['M2Mkey217.141014'] = { 'name': 'M2Mkey217.141014', 'color': 0, 'x': -0.205255, 'y': -0.044230, 'z': -0.180725 }
ellipses['M2Akey217.141030'] = { 'name': 'M2Akey217.141030', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.090079, 'y': -0.116932, 'z': -0.021377 }
points['M2Akey217.141030'] = { 'name': 'M2Akey217.141030', 'color': 0, 'x': 0.090079, 'y': -0.116932, 'z': -0.021377 }
ellipses['M2Rinr217.141027'] = { 'name': 'M2Rinr217.141027', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.058082, 'y': 0.301237, 'z': -0.003302 }
points['M2Rinr217.141027'] = { 'name': 'M2Rinr217.141027', 'color': 0, 'x': -0.058082, 'y': 0.301237, 'z': -0.003302 }
var segments = 16, rings = 16, radius = -0.011129
var xaxislength = 0.556452
var yaxislength = 0.420681
var zaxislength = 0.423130
var min_x = -0.205255
var min_y = -0.119445
var min_z = -0.180725
var max_x = 0.351197
var max_y = 0.301237
var max_z = 0.242406
max = 0.403876
pc1 = 24
pc2 = 8
pc3 = 2
"""
input_ellipses_dict_b= {'M3Gkey217.141037': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.15853553,  0.13626829,  0.00695479])}, 'M9Gkey217.141099': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.1330006 , -0.11902664,  0.03436203])}, 'M3Ekey217.141042': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.16537289,  0.12398874,  0.05474084])}, 'M3Nkey217.141001': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.22848809,  0.04261731,  0.14650494])}, 'M2Bkey217.141063': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.29978463, -0.02836534, -0.05165183])}, 'M9Nkey217.141010': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.24571407, -0.10429751,  0.06022179])}, 'M3Ykey217.141023': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.11876957,  0.16916626, -0.00046783])}, 'M2Lkey217.141048': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.20040415,  0.02287306,  0.02459859])}, 'M9Indl217.141066': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.04673168, -0.10235922, -0.21866154])}, 'M9Hkey217.140999': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.19022657, -0.19786448,  0.02729295])}, 'M3Rinl217.141100': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.17205451,  0.04193414,  0.00484723])}, 'M2Vkey217.141022': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.32245696, -0.00381042, -0.01360584])}, 'M9Kkey217.141033': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.22687595, -0.25175467,  0.1741462 ])}, 'M9Pinl217.141035': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.07892599,  0.01816048, -0.19335542])}, 'M2Rsft217.141034': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.30322032, -0.02264274,  0.01533071])}, 'M9Enter217.141084': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.21810166, -0.09168339,  0.0636011 ])}, 'M3Rinr217.141091': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.04527469,  0.1730041 , -0.08075781])}, 'M2Dkey217.140994': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.37533536, -0.09318311,  0.1122991 ])}, 'M9Bkey217.141032': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.24131931, -0.09030056, -0.05605445])}, 'M2Midr217.141082': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.19891053,  0.0242793 , -0.10971495])}, 'M2Ukey217.141085': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.1948824 ,  0.0271438 , -0.03586529])}, 'M9Ekey217.141044': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.18387734, -0.07883634,  0.06612579])}, 'M3Vkey217.141028': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.19118363,  0.0411992 ,  0.04642868])}, 'M9Wkey217.141090': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.20766412, -0.21006608,  0.03139356])}, 'M3Indr217.140993': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.0812147 ,  0.22270225,  0.01804675])}, 'M3Indl217.141067': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.14693955,  0.05461802, -0.0877571 ])}, 'M2Ikey217.141000': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.26092955,  0.06769369,  0.0229684 ])}, 'M2Ykey217.141095': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.23230424,  0.05930433, -0.0180131 ])}, 'M2Mkey217.141014': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.13353248,  0.01383893, -0.06173079])}, 'M9Xkey217.141072': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.23338162, -0.10103526, -0.0297163 ])}, 'M3Xkey217.141055': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.10836527,  0.19812994, -0.05087926])}, 'M2Akey217.141030': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.32491826, -0.03270744,  0.01712272])}, 'M3Mkey217.141107': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.13563408,  0.14233038,  0.12676463])}, 'M9Qkey217.141019': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.2355375 , -0.16224466,  0.06869653])}, 'M9Thmr217.141049': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.07157203, -0.03102851, -0.25995233])}, 'M3Midr217.141101': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.03154121,  0.22962595, -0.04235793])}, 'M2Enter217.141012': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.28128779,  0.02743989, -0.03168212])}, 'M3Space217.141098': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.21178684,  0.17013266,  0.14469238])}, 'M2Okey217.141061': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.19342275,  0.01629536, -0.01546397])}, 'M9Rinl217.141020': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.14545461, -0.01974886, -0.19010292])}, 'M2Lsft217.141056': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.24148315,  0.00637286, -0.05189544])}, 'M2Tkey217.141054': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.29618714, -0.00037197,  0.01093486])}, 'M3Kkey217.141046': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.20340399, -0.00284395,  0.16308252])}, 'M9Pkey217.141096': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.26464201, -0.12804791,  0.18812316])}, 'M9Midl217.141043': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.16584767, -0.10202112, -0.16337482])}, 'M3Tkey217.141081': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.1385123 ,  0.08300605, -0.01795985])}, 'M2Xkey217.141016': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.26132945, -0.1039092 ,  0.12625588])}, 'M2Pinr217.141039': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.15774233,  0.07446709, -0.05580419])}, 'M9Fkey217.141065': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.2288685, -0.1260656, -0.0437725])}, 'M9Ykey217.141029': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.26258725, -0.10259103,  0.07121653])}, 'M9Dkey217.141053': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.24226885, -0.13610761,  0.00729644])}, 'M3Zkey217.141083': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.22658494,  0.07683592,  0.23849395])}, 'M9Rinr217.141080': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.14236071, -0.10003758, -0.24641515])}, 'M2Indl217.141074': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.25616181, -0.01091806, -0.01946556])}, 'M9Pinr217.141002': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.11822615, -0.18154117, -0.10456419])}, 'M3Thmr217.140996': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.08647969,  0.24786741,  0.00568417])}, 'M2Rinr217.141027': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.21695148,  0.00148657, -0.08101313])}, 'M2Hkey217.141026': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.24342978,  0.00252124, -0.04452293])}, 'M2Gkey217.141106': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.32920394, -0.03172291, -0.0799145 ])}, 'M2Fkey217.141078': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.34247703,  0.00542538,  0.01392341])}, 'M2Space217.140997': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.32344841,  0.03503185,  0.00778372])}, 'M3Pinl217.141094': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.01102694,  0.19558598, -0.05574228])}, 'M2Rkey217.141050': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.25794581, -0.06773594,  0.09881551])}, 'M3Ckey217.141073': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.20965232,  0.0670544 ,  0.17816539])}, 'M2Kkey217.141041': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.25990518, -0.01435957, -0.0353077 ])}, 'M3Pkey217.141031': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.12947346,  0.13288428,  0.0932493 ])}, 'M3Hkey217.141093': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.1489323 ,  0.12396304,  0.09296346])}, 'M2Pinl217.141003': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.21967102, -0.069647  ,  0.03976326])}, 'M2Midl217.141009': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.26312565, -0.1115054 ,  0.1261963 ])}, 'M9Indr217.140998': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.16469333, -0.13837032, -0.24509822])}, 'M9Space217.141079': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.23534334, -0.14998978,  0.05983008])}, 'M9Thml217.141021': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.19002526, -0.16375666, -0.16746877])}, 'M3Bkey217.141059': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.17159802,  0.07718138,  0.02905725])}, 'M2Pkey217.141105': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.21886081,  0.00759636, -0.01102251])}, 'M3Rkey217.141025': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.10981008,  0.11552642, -0.07146573])}, 'M2Jkey217.141038': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.23771669,  0.03500605, -0.02977612])}, 'M3Wkey217.141104': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.14972666,  0.16344765, -0.01125727])}, 'M3Qkey217.141051': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.15272782,  0.09496936, -0.04342489])}, 'M3Lsft217.141088': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.1006203 ,  0.154945  ,  0.08789295])}, 'M2Indr217.141062': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.11245005,  0.05601527, -0.10776842])}, 'M9Okey217.141076': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.17152214, -0.13776205, -0.02879131])}, 'M3Thml217.141040': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.19207376,  0.04262992,  0.00372789])}, 'M2Ckey217.141092': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.40150096, -0.10432501,  0.11080734])}, 'M9Mkey217.141007': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.20562155, -0.05996472, -0.08431799])}, 'M3Jkey217.141017': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.19877393,  0.01073988,  0.0293375 ])}, 'M2Wkey217.141070': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.32755775, -0.0340003 ,  0.08040463])}, 'M9Midr217.141060': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.12489135, -0.06409587, -0.255399  ])}, 'M9Skey217.141004': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.24885434, -0.18124083,  0.09078715])}, 'M2Zkey217.141015': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.31510894,  0.00277162,  0.04503167])}, 'M2Thml217.141005': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.29595702, -0.09690115,  0.08541305])}, 'M2Nkey217.141052': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.16304793, -0.042688  , -0.0177601 ])}, 'M3Midl217.141103': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.09079776,  0.11843183, -0.06992126])}, 'M2Ekey217.141011': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.36097226, -0.09727965,  0.15418572])}, 'M2Rinl217.141058': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.22300562, -0.12349778,  0.11515095])}, 'M3Rsft217.140995': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.13640758,  0.13893371,  0.00320775])}, 'M3Akey217.141006': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.1736383 ,  0.15137109,  0.08063106])}, 'M3Pinr217.141057': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.11094472,  0.22616276,  0.02330851])}, 'M2Qkey217.141102': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.28087057,  0.00849966, -0.00247148])}, 'M9Vkey217.141018': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.14517307, -0.05032832, -0.05349597])}, 'M9Ckey217.141013': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.22324527, -0.08135402, -0.02142975])}, 'M2Thmr217.141087': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.14646651,  0.01911532, -0.13617111])}, 'M2Skey217.141089': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([ 0.28698731, -0.00351613,  0.03721589])}, 'M9Akey217.141086': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.15414947, -0.1333059 ,  0.12891129])}, 'M3Lkey217.141097': {'axes_radii': [0.0, 0.0, 0.0], 'center': array([-0.09995915,  0.19416634,  0.01059473])}}
input_mapping_file_tuple_b = ([['M2Thmr217.141087', 'AATCAGTCTCGT', 'CATGCTGCCTCCCGTAGGAGT', 'M2Thmr217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'human skin metagenome', '0', '539655', 'unassigned', 'UBERON:skin of finger', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:human-associated habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Thmr217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'UBERON:skin', 'Forensic_identification_using_skin_bacterial_communities', 'M2Thmr217_V2', 'ENVO:sebum', '16S rRNA', 'ENVO:human-associated habitat', 'TCAG', 'UBERON:sebum', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'finger_tip'], ['M2Akey217.141030', 'ACATGATCGTTC', 'CATGCTGCCTCCCGTAGGAGT', 'M2Akey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Akey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M2Akey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Akey'], ['M2Rinr217.141027', 'ACACATGTCTAC', 'CATGCTGCCTCCCGTAGGAGT', 'M2Rinr217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'human skin metagenome', '0', '539655', 'unassigned', 'UBERON:skin of finger', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:human-associated habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Rinr217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'UBERON:skin', 'Forensic_identification_using_skin_bacterial_communities', 'M2Rinr217_V2', 'ENVO:sebum', '16S rRNA', 'ENVO:human-associated habitat', 'TCAG', 'UBERON:sebum', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'finger_tip'], ['M2Mkey217.141014', 'ACGCTATCTGGA', 'CATGCTGCCTCCCGTAGGAGT', 'M2Mkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Mkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M2Mkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Mkey'], ['M2Nkey217.141052', 'ACGCGCAGATAC', 'CATGCTGCCTCCCGTAGGAGT', 'M2Nkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Nkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M2Nkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Nkey'], ['M2Midl217.141009', 'AACTGTGCGTAC', 'CATGCTGCCTCCCGTAGGAGT', 'M2Midl217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'human skin metagenome', '0', '539655', 'unassigned', 'UBERON:skin of finger', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:human-associated habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Midl217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'UBERON:skin', 'Forensic_identification_using_skin_bacterial_communities', 'M2Midl217_V2', 'ENVO:sebum', '16S rRNA', 'ENVO:human-associated habitat', 'TCAG', 'UBERON:sebum', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'finger_tip'], ['M2Ukey217.141085', 'ACAGCTAGCTTG', 'CATGCTGCCTCCCGTAGGAGT', 'M2Ukey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Ukey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M2Ukey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Ukey'], ['M2Zkey217.141015', 'ACGACGTCTTAG', 'CATGCTGCCTCCCGTAGGAGT', 'M2Zkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Zkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M2Zkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Zkey'], ['M2Kkey217.141041', 'ACCTCGATCAGA', 'CATGCTGCCTCCCGTAGGAGT', 'M2Kkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Kkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M2Kkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Kkey'], ['M2Enter217.141012', 'ACGGTGAGTGTC', 'CATGCTGCCTCCCGTAGGAGT', 'M2Enter217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Enter217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M2Enter217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Ente'], ['M2Jkey217.141038', 'ACCGCAGAGTCA', 'CATGCTGCCTCCCGTAGGAGT', 'M2Jkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Jkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M2Jkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Jkey'], ['M2Midr217.141082', 'ACACACTATGGC', 'CATGCTGCCTCCCGTAGGAGT', 'M2Midr217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'human skin metagenome', '0', '539655', 'unassigned', 'UBERON:skin of finger', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:human-associated habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Midr217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'UBERON:skin', 'Forensic_identification_using_skin_bacterial_communities', 'M2Midr217_V2', 'ENVO:sebum', '16S rRNA', 'ENVO:human-associated habitat', 'TCAG', 'UBERON:sebum', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'finger_tip'], ['M2Ekey217.141011', 'ACACTGTTCATG', 'CATGCTGCCTCCCGTAGGAGT', 'M2Ekey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Ekey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M2Ekey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Ekey'], ['M2Hkey217.141026', 'ACCAGCGACTAG', 'CATGCTGCCTCCCGTAGGAGT', 'M2Hkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Hkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M2Hkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Hkey'], ['M2Okey217.141061', 'ACAGTTGCGCGA', 'CATGCTGCCTCCCGTAGGAGT', 'M2Okey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Okey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M2Okey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Okey'], ['M2Bkey217.141063', 'ACGCGATACTGG', 'CATGCTGCCTCCCGTAGGAGT', 'M2Bkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Bkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M2Bkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Bkey'], ['M2Skey217.141089', 'ACATGTCACGTG', 'CATGCTGCCTCCCGTAGGAGT', 'M2Skey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Skey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M2Skey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Skey'], ['M2Ckey217.141092', 'ACGATGCGACCA', 'CATGCTGCCTCCCGTAGGAGT', 'M2Ckey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Ckey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M2Ckey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Ckey'], ['M2Indr217.141062', 'AATCGTGACTCG', 'CATGCTGCCTCCCGTAGGAGT', 'M2Indr217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'human skin metagenome', '0', '539655', 'unassigned', 'UBERON:skin of finger', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:human-associated habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Indr217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'UBERON:skin', 'Forensic_identification_using_skin_bacterial_communities', 'M2Indr217_V2', 'ENVO:sebum', '16S rRNA', 'ENVO:human-associated habitat', 'TCAG', 'UBERON:sebum', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'finger_tip'], ['M2Gkey217.141106', 'ACCAGACGATGC', 'CATGCTGCCTCCCGTAGGAGT', 'M2Gkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Gkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M2Gkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Gkey'], ['M2Space217.140997', 'ACGTACTCAGTG', 'CATGCTGCCTCCCGTAGGAGT', 'M2Space217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Space217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M2Space217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Space_bar'], ['M2Lsft217.141056', 'ACGGATCGTCAG', 'CATGCTGCCTCCCGTAGGAGT', 'M2Lsft217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Lsft217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M2Lsft217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Left_shift'], ['M2Pkey217.141105', 'ACATCACTTAGC', 'CATGCTGCCTCCCGTAGGAGT', 'M2Pkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Pkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M2Pkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Pkey'], ['M2Tkey217.141054', 'ACAGAGTCGGCT', 'CATGCTGCCTCCCGTAGGAGT', 'M2Tkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Tkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M2Tkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Tkey'], ['M2Pinl217.141003', 'AAGCTGCAGTCG', 'CATGCTGCCTCCCGTAGGAGT', 'M2Pinl217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'human skin metagenome', '0', '539655', 'unassigned', 'UBERON:skin of finger', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:human-associated habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Pinl217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'UBERON:skin', 'Forensic_identification_using_skin_bacterial_communities', 'M2Pinl217_V2', 'ENVO:sebum', '16S rRNA', 'ENVO:human-associated habitat', 'TCAG', 'UBERON:sebum', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'finger_tip'], ['M2Vkey217.141022', 'ACGCAACTGCTA', 'CATGCTGCCTCCCGTAGGAGT', 'M2Vkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Vkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M2Vkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Vkey'], ['M2Rinl217.141058', 'AAGAGATGTCGA', 'CATGCTGCCTCCCGTAGGAGT', 'M2Rinl217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'human skin metagenome', '0', '539655', 'unassigned', 'UBERON:skin of finger', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:human-associated habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Rinl217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'UBERON:skin', 'Forensic_identification_using_skin_bacterial_communities', 'M2Rinl217_V2', 'ENVO:sebum', '16S rRNA', 'ENVO:human-associated habitat', 'TCAG', 'UBERON:sebum', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'finger_tip'], ['M2Indl217.141074', 'AACTCGTCGATG', 'CATGCTGCCTCCCGTAGGAGT', 'M2Indl217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'human skin metagenome', '0', '539655', 'unassigned', 'UBERON:skin of finger', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:human-associated habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Indl217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'UBERON:skin', 'Forensic_identification_using_skin_bacterial_communities', 'M2Indl217_V2', 'ENVO:sebum', '16S rRNA', 'ENVO:human-associated habitat', 'TCAG', 'UBERON:sebum', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'finger_tip'], ['M2Rkey217.141050', 'ACAGACCACTCA', 'CATGCTGCCTCCCGTAGGAGT', 'M2Rkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Rkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M2Rkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Rkey'], ['M2Fkey217.141078', 'ACCACATACATC', 'CATGCTGCCTCCCGTAGGAGT', 'M2Fkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Fkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M2Fkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Fkey'], ['M2Thml217.141005', 'AACGCACGCTAG', 'CATGCTGCCTCCCGTAGGAGT', 'M2Thml217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'human skin metagenome', '0', '539655', 'unassigned', 'UBERON:skin of finger', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:human-associated habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Thml217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'UBERON:skin', 'Forensic_identification_using_skin_bacterial_communities', 'M2Thml217_V2', 'ENVO:sebum', '16S rRNA', 'ENVO:human-associated habitat', 'TCAG', 'UBERON:sebum', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'finger_tip'], ['M2Dkey217.140994', 'ACATTCAGCGCA', 'CATGCTGCCTCCCGTAGGAGT', 'M2Dkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Dkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M2Dkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Dkey'], ['M2Ykey217.141095', 'ACAGCAGTGGTC', 'CATGCTGCCTCCCGTAGGAGT', 'M2Ykey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Ykey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M2Ykey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Ykey'], ['M2Ikey217.141000', 'ACAGTGCTTCAT', 'CATGCTGCCTCCCGTAGGAGT', 'M2Ikey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Ikey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M2Ikey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Ikey'], ['M2Wkey217.141070', 'ACACTAGATCCG', 'CATGCTGCCTCCCGTAGGAGT', 'M2Wkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Wkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M2Wkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Wkey'], ['M2Rsft217.141034', 'ACGCTCATGGAT', 'CATGCTGCCTCCCGTAGGAGT', 'M2Rsft217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Rsft217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M2Rsft217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Right_shift'], ['M2Lkey217.141048', 'ACCTGTCTCTCT', 'CATGCTGCCTCCCGTAGGAGT', 'M2Lkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Lkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M2Lkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Lkey'], ['M2Qkey217.141102', 'ACACGGTGTCTA', 'CATGCTGCCTCCCGTAGGAGT', 'M2Qkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Qkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M2Qkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Qkey'], ['M2Pinr217.141039', 'ACACGAGCCACA', 'CATGCTGCCTCCCGTAGGAGT', 'M2Pinr217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'human skin metagenome', '0', '539655', 'unassigned', 'UBERON:skin of finger', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:human-associated habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Pinr217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'UBERON:skin', 'Forensic_identification_using_skin_bacterial_communities', 'M2Pinr217_V2', 'ENVO:sebum', '16S rRNA', 'ENVO:human-associated habitat', 'TCAG', 'UBERON:sebum', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'finger_tip'], ['M2Xkey217.141016', 'ACGAGTGCTATC', 'CATGCTGCCTCCCGTAGGAGT', 'M2Xkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '36', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M2', 'Yes', 'M2Xkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M2Xkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '36', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Xkey'], ['M3Rinr217.141091', 'AGGTGTGATCGC', 'CATGCTGCCTCCCGTAGGAGT', 'M3Rinr217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '33', 'human skin metagenome', '0', '539655', 'unassigned', 'UBERON:skin of finger', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:human-associated habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M3', 'Yes', 'M3Rinr217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'UBERON:skin', 'Forensic_identification_using_skin_bacterial_communities', 'M3Rinr217_V2', 'ENVO:sebum', '16S rRNA', 'ENVO:human-associated habitat', 'TCAG', 'UBERON:sebum', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '33', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'finger_tip'], ['M3Vkey217.141028', 'ATATCGCTACTG', 'CATGCTGCCTCCCGTAGGAGT', 'M3Vkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '33', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M3', 'Yes', 'M3Vkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M3Vkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '33', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Vkey'], ['M3Hkey217.141093', 'ATACACGTGGCG', 'CATGCTGCCTCCCGTAGGAGT', 'M3Hkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '33', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M3', 'Yes', 'M3Hkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M3Hkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '33', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Hkey'], ['M3Rkey217.141025', 'AGTCCATAGCTG', 'CATGCTGCCTCCCGTAGGAGT', 'M3Rkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '33', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M3', 'Yes', 'M3Rkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M3Rkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '33', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Rkey'], ['M3Midl217.141103', 'AGCTCCATACAG', 'CATGCTGCCTCCCGTAGGAGT', 'M3Midl217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '33', 'human skin metagenome', '0', '539655', 'unassigned', 'UBERON:skin of finger', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:human-associated habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M3', 'Yes', 'M3Midl217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'UBERON:skin', 'Forensic_identification_using_skin_bacterial_communities', 'M3Midl217_V2', 'ENVO:sebum', '16S rRNA', 'ENVO:human-associated habitat', 'TCAG', 'UBERON:sebum', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '33', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'finger_tip'], ['M3Tkey217.141081', 'AGTCTACTCTGA', 'CATGCTGCCTCCCGTAGGAGT', 'M3Tkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '33', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M3', 'Yes', 'M3Tkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M3Tkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '33', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Tkey'], ['M3Wkey217.141104', 'AGTAGTATCCTC', 'CATGCTGCCTCCCGTAGGAGT', 'M3Wkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '33', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M3', 'Yes', 'M3Wkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M3Wkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '33', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Wkey'], ['M3Jkey217.141017', 'ATACAGAGCTCC', 'CATGCTGCCTCCCGTAGGAGT', 'M3Jkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '33', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M3', 'Yes', 'M3Jkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M3Jkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '33', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Jkey'], ['M3Indr217.140993', 'AGGACGCACTGT', 'CATGCTGCCTCCCGTAGGAGT', 'M3Indr217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '33', 'human skin metagenome', '0', '539655', 'unassigned', 'UBERON:skin of finger', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:human-associated habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M3', 'Yes', 'M3Indr217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'UBERON:skin', 'Forensic_identification_using_skin_bacterial_communities', 'M3Indr217_V2', 'ENVO:sebum', '16S rRNA', 'ENVO:human-associated habitat', 'TCAG', 'UBERON:sebum', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '33', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'finger_tip'], ['M3Ekey217.141042', 'AGTCACATCACT', 'CATGCTGCCTCCCGTAGGAGT', 'M3Ekey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '33', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M3', 'Yes', 'M3Ekey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M3Ekey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '33', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Ekey'], ['M3Nkey217.141001', 'ATCACGTAGCGG', 'CATGCTGCCTCCCGTAGGAGT', 'M3Nkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '33', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M3', 'Yes', 'M3Nkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M3Nkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '33', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Nkey'], ['M3Indl217.141067', 'AGCTATCCACGA', 'CATGCTGCCTCCCGTAGGAGT', 'M3Indl217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '33', 'human skin metagenome', '0', '539655', 'unassigned', 'UBERON:skin of finger', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:human-associated habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M3', 'Yes', 'M3Indl217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'UBERON:skin', 'Forensic_identification_using_skin_bacterial_communities', 'M3Indl217_V2', 'ENVO:sebum', '16S rRNA', 'ENVO:human-associated habitat', 'TCAG', 'UBERON:sebum', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '33', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'finger_tip'], ['M3Space217.141098', 'ATCGATCTGTGG', 'CATGCTGCCTCCCGTAGGAGT', 'M3Space217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '33', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M3', 'Yes', 'M3Space217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M3Space217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '33', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Space_bar'], ['M3Gkey217.141037', 'ATAATCTCGTCG', 'CATGCTGCCTCCCGTAGGAGT', 'M3Gkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '33', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M3', 'Yes', 'M3Gkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M3Gkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '33', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Gkey'], ['M3Mkey217.141107', 'ATCACTAGTCAC', 'CATGCTGCCTCCCGTAGGAGT', 'M3Mkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '33', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M3', 'Yes', 'M3Mkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M3Mkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '33', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Mkey'], ['M3Ykey217.141023', 'AGTCTCGCATAT', 'CATGCTGCCTCCCGTAGGAGT', 'M3Ykey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '33', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M3', 'Yes', 'M3Ykey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M3Ykey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '33', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Ykey'], ['M3Pinl217.141094', 'AGCTGACTAGTC', 'CATGCTGCCTCCCGTAGGAGT', 'M3Pinl217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '33', 'human skin metagenome', '0', '539655', 'unassigned', 'UBERON:skin of finger', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:human-associated habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M3', 'Yes', 'M3Pinl217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'UBERON:skin', 'Forensic_identification_using_skin_bacterial_communities', 'M3Pinl217_V2', 'ENVO:sebum', '16S rRNA', 'ENVO:human-associated habitat', 'TCAG', 'UBERON:sebum', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '33', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'finger_tip'], ['M3Midr217.141101', 'AGGCTACACGAC', 'CATGCTGCCTCCCGTAGGAGT', 'M3Midr217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '33', 'human skin metagenome', '0', '539655', 'unassigned', 'UBERON:skin of finger', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:human-associated habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M3', 'Yes', 'M3Midr217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'UBERON:skin', 'Forensic_identification_using_skin_bacterial_communities', 'M3Midr217_V2', 'ENVO:sebum', '16S rRNA', 'ENVO:human-associated habitat', 'TCAG', 'UBERON:sebum', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '33', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'finger_tip'], ['M3Rsft217.140995', 'ATCCGATCACAG', 'CATGCTGCCTCCCGTAGGAGT', 'M3Rsft217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '33', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M3', 'Yes', 'M3Rsft217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M3Rsft217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '33', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Right_shift'], ['M3Thml217.141040', 'AGCGTAGGTCGT', 'CATGCTGCCTCCCGTAGGAGT', 'M3Thml217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '33', 'human skin metagenome', '0', '539655', 'unassigned', 'UBERON:skin of finger', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:human-associated habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M3', 'Yes', 'M3Thml217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'UBERON:skin', 'Forensic_identification_using_skin_bacterial_communities', 'M3Thml217_V2', 'ENVO:sebum', '16S rRNA', 'ENVO:human-associated habitat', 'TCAG', 'UBERON:sebum', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '33', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'finger_tip'], ['M3Xkey217.141055', 'ATAGCTCCATAC', 'CATGCTGCCTCCCGTAGGAGT', 'M3Xkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '33', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M3', 'Yes', 'M3Xkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M3Xkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '33', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Xkey'], ['M3Ckey217.141073', 'ATAGGCGATCTC', 'CATGCTGCCTCCCGTAGGAGT', 'M3Ckey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '33', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M3', 'Yes', 'M3Ckey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M3Ckey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '33', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Ckey'], ['M3Qkey217.141051', 'AGTACTGCAGGC', 'CATGCTGCCTCCCGTAGGAGT', 'M3Qkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '33', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M3', 'Yes', 'M3Qkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M3Qkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '33', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Qkey'], ['M3Pkey217.141031', 'AGTGTCACGGTG', 'CATGCTGCCTCCCGTAGGAGT', 'M3Pkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '33', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M3', 'Yes', 'M3Pkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M3Pkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '33', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Pkey'], ['M3Rinl217.141100', 'AGCTCTCAGAGG', 'CATGCTGCCTCCCGTAGGAGT', 'M3Rinl217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '33', 'human skin metagenome', '0', '539655', 'unassigned', 'UBERON:skin of finger', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:human-associated habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M3', 'Yes', 'M3Rinl217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'UBERON:skin', 'Forensic_identification_using_skin_bacterial_communities', 'M3Rinl217_V2', 'ENVO:sebum', '16S rRNA', 'ENVO:human-associated habitat', 'TCAG', 'UBERON:sebum', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '33', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'finger_tip'], ['M3Lkey217.141097', 'ATACTATTGCGC', 'CATGCTGCCTCCCGTAGGAGT', 'M3Lkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '33', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M3', 'Yes', 'M3Lkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M3Lkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '33', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Lkey'], ['M3Thmr217.140996', 'AGCTTGACAGCT', 'CATGCTGCCTCCCGTAGGAGT', 'M3Thmr217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '33', 'human skin metagenome', '0', '539655', 'unassigned', 'UBERON:skin of finger', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:human-associated habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M3', 'Yes', 'M3Thmr217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'UBERON:skin', 'Forensic_identification_using_skin_bacterial_communities', 'M3Thmr217_V2', 'ENVO:sebum', '16S rRNA', 'ENVO:human-associated habitat', 'TCAG', 'UBERON:sebum', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '33', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'finger_tip'], ['M3Bkey217.141059', 'ATATGCCAGTGC', 'CATGCTGCCTCCCGTAGGAGT', 'M3Bkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '33', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M3', 'Yes', 'M3Bkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M3Bkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '33', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Bkey'], ['M3Akey217.141006', 'AGTGTTCGATCG', 'CATGCTGCCTCCCGTAGGAGT', 'M3Akey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '33', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M3', 'Yes', 'M3Akey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M3Akey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '33', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Akey'], ['M3Lsft217.141088', 'ATCAGGCGTGTG', 'CATGCTGCCTCCCGTAGGAGT', 'M3Lsft217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '33', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M3', 'Yes', 'M3Lsft217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M3Lsft217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '33', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Left_Shift'], ['M3Kkey217.141046', 'ATACGTCTTCGA', 'CATGCTGCCTCCCGTAGGAGT', 'M3Kkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '33', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M3', 'Yes', 'M3Kkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M3Kkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '33', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Kkey'], ['M3Pinr217.141057', 'AGTACGCTCGAG', 'CATGCTGCCTCCCGTAGGAGT', 'M3Pinr217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '33', 'human skin metagenome', '0', '539655', 'unassigned', 'UBERON:skin of finger', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:human-associated habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M3', 'Yes', 'M3Pinr217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'UBERON:skin', 'Forensic_identification_using_skin_bacterial_communities', 'M3Pinr217_V2', 'ENVO:sebum', '16S rRNA', 'ENVO:human-associated habitat', 'TCAG', 'UBERON:sebum', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '33', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'finger_tip'], ['M3Zkey217.141083', 'ATACTCACTCAG', 'CATGCTGCCTCCCGTAGGAGT', 'M3Zkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '33', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M3', 'Yes', 'M3Zkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M3Zkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '33', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Zkey'], ['M9Pinl217.141035', 'ACTACAGCCTAT', 'CATGCTGCCTCCCGTAGGAGT', 'M9Pinl217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '25', 'human skin metagenome', '0', '539655', 'unassigned', 'UBERON:skin of finger', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:human-associated habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M9', 'Yes', 'M9Pinl217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'UBERON:skin', 'Forensic_identification_using_skin_bacterial_communities', 'M9Pinl217_V2', 'ENVO:sebum', '16S rRNA', 'ENVO:human-associated habitat', 'TCAG', 'UBERON:sebum', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '25', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'finger_tip'], ['M9Enter217.141084', 'AGCGAGCTATCT', 'CATGCTGCCTCCCGTAGGAGT', 'M9Enter217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '25', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M9', 'Yes', 'M9Enter217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M9Enter217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '25', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Ente'], ['M9Thmr217.141049', 'ACTACGTGTGGT', 'CATGCTGCCTCCCGTAGGAGT', 'M9Thmr217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '25', 'human skin metagenome', '0', '539655', 'unassigned', 'UBERON:skin of finger', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:human-associated habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M9', 'Yes', 'M9Thmr217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'UBERON:skin', 'Forensic_identification_using_skin_bacterial_communities', 'M9Thmr217_V2', 'ENVO:sebum', '16S rRNA', 'ENVO:human-associated habitat', 'TCAG', 'UBERON:sebum', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '25', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'finger_tip'], ['M9Thml217.141021', 'ACGTCTGTAGCA', 'CATGCTGCCTCCCGTAGGAGT', 'M9Thml217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '25', 'human skin metagenome', '0', '539655', 'unassigned', 'UBERON:skin of finger', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:human-associated habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M9', 'Yes', 'M9Thml217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'UBERON:skin', 'Forensic_identification_using_skin_bacterial_communities', 'M9Thml217_V2', 'ENVO:sebum', '16S rRNA', 'ENVO:human-associated habitat', 'TCAG', 'UBERON:sebum', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '25', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'finger_tip'], ['M9Midr217.141060', 'ACTATTGTCACG', 'CATGCTGCCTCCCGTAGGAGT', 'M9Midr217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '25', 'human skin metagenome', '0', '539655', 'unassigned', 'UBERON:skin of finger', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:human-associated habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M9', 'Yes', 'M9Midr217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'UBERON:skin', 'Forensic_identification_using_skin_bacterial_communities', 'M9Midr217_V2', 'ENVO:sebum', '16S rRNA', 'ENVO:human-associated habitat', 'TCAG', 'UBERON:sebum', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '25', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'finger_tip'], ['M9Qkey217.141019', 'ACTCGATTCGAT', 'CATGCTGCCTCCCGTAGGAGT', 'M9Qkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '25', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M9', 'Yes', 'M9Qkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M9Qkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '25', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Qkey'], ['M9Indl217.141066', 'ACGTGAGAGAAT', 'CATGCTGCCTCCCGTAGGAGT', 'M9Indl217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '25', 'human skin metagenome', '0', '539655', 'unassigned', 'UBERON:skin of finger', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:human-associated habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M9', 'Yes', 'M9Indl217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'UBERON:skin', 'Forensic_identification_using_skin_bacterial_communities', 'M9Indl217_V2', 'ENVO:sebum', '16S rRNA', 'ENVO:human-associated habitat', 'TCAG', 'UBERON:sebum', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '25', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'finger_tip'], ['M9Rinl217.141020', 'ACGTTAGCACAC', 'CATGCTGCCTCCCGTAGGAGT', 'M9Rinl217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '25', 'human skin metagenome', '0', '539655', 'unassigned', 'UBERON:skin of finger', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:human-associated habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M9', 'Yes', 'M9Rinl217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'UBERON:skin', 'Forensic_identification_using_skin_bacterial_communities', 'M9Rinl217_V2', 'ENVO:sebum', '16S rRNA', 'ENVO:human-associated habitat', 'TCAG', 'UBERON:sebum', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '25', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'finger_tip'], ['M9Gkey217.141099', 'AGAGCAAGAGCA', 'CATGCTGCCTCCCGTAGGAGT', 'M9Gkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '25', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M9', 'Yes', 'M9Gkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M9Gkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '25', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Gkey'], ['M9Ckey217.141013', 'AGCACACCTACA', 'CATGCTGCCTCCCGTAGGAGT', 'M9Ckey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '25', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M9', 'Yes', 'M9Ckey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M9Ckey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '25', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Ckey'], ['M9Okey217.141076', 'ACTTGTAGCAGC', 'CATGCTGCCTCCCGTAGGAGT', 'M9Okey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '25', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M9', 'Yes', 'M9Okey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M9Okey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '25', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Okey'], ['M9Midl217.141043', 'ACGTGCCGTAGA', 'CATGCTGCCTCCCGTAGGAGT', 'M9Midl217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '25', 'human skin metagenome', '0', '539655', 'unassigned', 'UBERON:skin of finger', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:human-associated habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M9', 'Yes', 'M9Midl217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'UBERON:skin', 'Forensic_identification_using_skin_bacterial_communities', 'M9Midl217_V2', 'ENVO:sebum', '16S rRNA', 'ENVO:human-associated habitat', 'TCAG', 'UBERON:sebum', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '25', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'finger_tip'], ['M9Space217.141079', 'AGCGCTGATGTG', 'CATGCTGCCTCCCGTAGGAGT', 'M9Space217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '25', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M9', 'Yes', 'M9Space217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M9Space217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '25', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Space_bar'], ['M9Hkey217.140999', 'AGAGTAGCTAAG', 'CATGCTGCCTCCCGTAGGAGT', 'M9Hkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '25', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M9', 'Yes', 'M9Hkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M9Hkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '25', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Hkey'], ['M9Ykey217.141029', 'ACTGTACGCGTA', 'CATGCTGCCTCCCGTAGGAGT', 'M9Ykey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '25', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M9', 'Yes', 'M9Ykey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M9Ykey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '25', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Ykey'], ['M9Akey217.141086', 'AGACCGTCAGAC', 'CATGCTGCCTCCCGTAGGAGT', 'M9Akey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '25', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M9', 'Yes', 'M9Akey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M9Akey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '25', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Akey'], ['M9Mkey217.141007', 'AGCATATGAGAG', 'CATGCTGCCTCCCGTAGGAGT', 'M9Mkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '25', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M9', 'Yes', 'M9Mkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M9Mkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '25', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Mkey'], ['M9Xkey217.141072', 'AGATGTTCTGCT', 'CATGCTGCCTCCCGTAGGAGT', 'M9Xkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '25', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M9', 'Yes', 'M9Xkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M9Xkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '25', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Xkey'], ['M9Ekey217.141044', 'ACTCTTCTAGAG', 'CATGCTGCCTCCCGTAGGAGT', 'M9Ekey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '25', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M9', 'Yes', 'M9Ekey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M9Ekey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '25', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Ekey'], ['M9Indr217.140998', 'ACTAGCTCCATA', 'CATGCTGCCTCCCGTAGGAGT', 'M9Indr217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '25', 'human skin metagenome', '0', '539655', 'unassigned', 'UBERON:skin of finger', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:human-associated habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M9', 'Yes', 'M9Indr217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'UBERON:skin', 'Forensic_identification_using_skin_bacterial_communities', 'M9Indr217_V2', 'ENVO:sebum', '16S rRNA', 'ENVO:human-associated habitat', 'TCAG', 'UBERON:sebum', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '25', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'finger_tip'], ['M9Vkey217.141018', 'AGCACGAGCCTA', 'CATGCTGCCTCCCGTAGGAGT', 'M9Vkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '25', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M9', 'Yes', 'M9Vkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M9Vkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '25', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Vkey'], ['M9Dkey217.141053', 'AGACTGCGTACT', 'CATGCTGCCTCCCGTAGGAGT', 'M9Dkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '25', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M9', 'Yes', 'M9Dkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M9Dkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '25', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Dkey'], ['M9Bkey217.141032', 'AGCAGCACTTGT', 'CATGCTGCCTCCCGTAGGAGT', 'M9Bkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '25', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M9', 'Yes', 'M9Bkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M9Bkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '25', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Bkey'], ['M9Rinr217.141080', 'ACTCACGGTATG', 'CATGCTGCCTCCCGTAGGAGT', 'M9Rinr217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '25', 'human skin metagenome', '0', '539655', 'unassigned', 'UBERON:skin of finger', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:human-associated habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M9', 'Yes', 'M9Rinr217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'UBERON:skin', 'Forensic_identification_using_skin_bacterial_communities', 'M9Rinr217_V2', 'ENVO:sebum', '16S rRNA', 'ENVO:human-associated habitat', 'TCAG', 'UBERON:sebum', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '25', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'finger_tip'], ['M9Kkey217.141033', 'AGATACACGCGC', 'CATGCTGCCTCCCGTAGGAGT', 'M9Kkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '25', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M9', 'Yes', 'M9Kkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M9Kkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '25', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Kkey'], ['M9Skey217.141004', 'AGACGTGCACTG', 'CATGCTGCCTCCCGTAGGAGT', 'M9Skey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '25', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M9', 'Yes', 'M9Skey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M9Skey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '25', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Skey'], ['M9Pkey217.141096', 'AGAACACGTCTC', 'CATGCTGCCTCCCGTAGGAGT', 'M9Pkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '25', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M9', 'Yes', 'M9Pkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M9Pkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '25', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Pkey'], ['M9Nkey217.141010', 'AGCAGTCGCGAT', 'CATGCTGCCTCCCGTAGGAGT', 'M9Nkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '25', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M9', 'Yes', 'M9Nkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M9Nkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '25', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Nkey'], ['M9Fkey217.141065', 'AGAGAGCAAGTG', 'CATGCTGCCTCCCGTAGGAGT', 'M9Fkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '25', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M9', 'Yes', 'M9Fkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M9Fkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '25', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Fkey'], ['M9Wkey217.141090', 'ACTCGCACAGGA', 'CATGCTGCCTCCCGTAGGAGT', 'M9Wkey217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '25', 'computer keyboard metagenome', '0', '256318', 'unassigned', 'NA', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:anthropogenic habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M9', 'Yes', 'M9Wkey217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'NA', 'Forensic_identification_using_skin_bacterial_communities', 'M9Wkey217_V2', 'ENVO:surface', '16S rRNA', 'ENVO:surface', 'TCAG', 'NA', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '25', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'Wkey'], ['M9Pinr217.141002', 'ACTCAGATACTC', 'CATGCTGCCTCCCGTAGGAGT', 'M9Pinr217_V2', 'V2', 'n', '8/14/08', 'Forensic_identification_using_skin_bacterial_communities', '25', 'human skin metagenome', '0', '539655', 'unassigned', 'UBERON:skin of finger', '1624', 'CCME', '7/15/08', '0', 'FFCKVMW', 'ENVO:human-associated habitat', 'male', 'FLX', 'Forensic_identification_using_skin_bacterial_communities', 'NA', 'M9', 'Yes', 'M9Pinr217', '0.8', 'CCME', '1, swab', '-105.2705', '232', 'fierer_forensic_keyboard', 'years', 'UBERON:skin', 'Forensic_identification_using_skin_bacterial_communities', 'M9Pinr217_V2', 'ENVO:sebum', '16S rRNA', 'ENVO:human-associated habitat', 'TCAG', 'UBERON:sebum', 'GAZ:United States of America', '16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).', '0', 'CCME', 'V2', 'FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT', '25', 'pyrosequencing', 'NA', 'Forensic_identification_using_skin_bacterial_communities', '40.0083', 'Forensic_identification_using_skin_bacterial_communities', 'y', 'CCME', 'finger_tip']], ['SampleID', 'BarcodeSequence', 'LinkerPrimerSequence', 'BARCODE_READ_GROUP_TAG', 'TARGET_SUBFRAGMENT', 'ASSIGNED_FROM_GEO', 'RUN_DATE', 'TITLE', 'AGE', 'COMMON_NAME', 'DEPTH', 'TAXON_ID', 'HOST_COMMON_NAME', 'BODY_SITE', 'ELEVATION', 'EXPERIMENT_CENTER', 'COLLECTION_DATE', 'ALTITUDE', 'RUN_PREFIX', 'ENV_BIOME', 'SEX', 'PLATFORM', 'RUN_ALIAS', 'POOL_MEMBER_ACCESSION', 'HOST_SUBJECT_ID', 'Used', 'ANONYMIZED_NAME', 'POOL_PROPORTION', 'SAMPLE_CENTER', 'SAMP_SIZE', 'LONGITUDE', 'STUDY_ID', 'STUDY_REF', 'AGE_UNIT', 'BODY_HABITAT', 'EXPERIMENT_ALIAS', 'POOL_MEMBER_NAME', 'ENV_MATTER', 'TARGET_GENE', 'ENV_FEATURE', 'KEY_SEQ', 'BODY_PRODUCT', 'COUNTRY', 'LIBRARY_CONSTRUCTION_PROTOCOL', 'REGION', 'RUN_CENTER', 'PRIMER_READ_GROUP_TAG', 'PCR_PRIMERS', 'AGE_IN_YEARS', 'SEQUENCING_METH', 'INSTRUMENT_NAME', 'EXPERIMENT_TITLE', 'LATITUDE', 'EXPERIMENT_DESIGN_DESCRIPTION', 'PUBLIC', 'STUDY_CENTER', 'Description']) 
input_pcoalabels_b = [27.5,12.2,7.4]
expected_output_webgl_string_b = """var headers = ["SampleID","BarcodeSequence","LinkerPrimerSequence","BARCODE_READ_GROUP_TAG","TARGET_SUBFRAGMENT","ASSIGNED_FROM_GEO","RUN_DATE","TITLE","AGE","COMMON_NAME","DEPTH","TAXON_ID","HOST_COMMON_NAME","BODY_SITE","ELEVATION","EXPERIMENT_CENTER","COLLECTION_DATE","ALTITUDE","RUN_PREFIX","ENV_BIOME","SEX","PLATFORM","RUN_ALIAS","POOL_MEMBER_ACCESSION","HOST_SUBJECT_ID","Used","ANONYMIZED_NAME","POOL_PROPORTION","SAMPLE_CENTER","SAMP_SIZE","LONGITUDE","STUDY_ID","STUDY_REF","AGE_UNIT","BODY_HABITAT","EXPERIMENT_ALIAS","POOL_MEMBER_NAME","ENV_MATTER","TARGET_GENE","ENV_FEATURE","KEY_SEQ","BODY_PRODUCT","COUNTRY","LIBRARY_CONSTRUCTION_PROTOCOL","REGION","RUN_CENTER","PRIMER_READ_GROUP_TAG","PCR_PRIMERS","AGE_IN_YEARS","SEQUENCING_METH","INSTRUMENT_NAME","EXPERIMENT_TITLE","LATITUDE","EXPERIMENT_DESIGN_DESCRIPTION","PUBLIC","STUDY_CENTER","Description"]
var mapping = {"M2Thmr217.141087": ["M2Thmr217.141087","AATCAGTCTCGT","CATGCTGCCTCCCGTAGGAGT","M2Thmr217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","human skin metagenome","0","539655","unassigned","UBERON:skin of finger","1624","CCME","7/15/08","0","FFCKVMW","ENVO:human-associated habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Thmr217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","UBERON:skin","Forensic_identification_using_skin_bacterial_communities","M2Thmr217_V2","ENVO:sebum","16S rRNA","ENVO:human-associated habitat","TCAG","UBERON:sebum","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","finger_tip"],"M2Akey217.141030": ["M2Akey217.141030","ACATGATCGTTC","CATGCTGCCTCCCGTAGGAGT","M2Akey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Akey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M2Akey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Akey"],"M2Rinr217.141027": ["M2Rinr217.141027","ACACATGTCTAC","CATGCTGCCTCCCGTAGGAGT","M2Rinr217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","human skin metagenome","0","539655","unassigned","UBERON:skin of finger","1624","CCME","7/15/08","0","FFCKVMW","ENVO:human-associated habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Rinr217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","UBERON:skin","Forensic_identification_using_skin_bacterial_communities","M2Rinr217_V2","ENVO:sebum","16S rRNA","ENVO:human-associated habitat","TCAG","UBERON:sebum","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","finger_tip"],"M2Mkey217.141014": ["M2Mkey217.141014","ACGCTATCTGGA","CATGCTGCCTCCCGTAGGAGT","M2Mkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Mkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M2Mkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Mkey"],"M2Nkey217.141052": ["M2Nkey217.141052","ACGCGCAGATAC","CATGCTGCCTCCCGTAGGAGT","M2Nkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Nkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M2Nkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Nkey"],"M2Midl217.141009": ["M2Midl217.141009","AACTGTGCGTAC","CATGCTGCCTCCCGTAGGAGT","M2Midl217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","human skin metagenome","0","539655","unassigned","UBERON:skin of finger","1624","CCME","7/15/08","0","FFCKVMW","ENVO:human-associated habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Midl217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","UBERON:skin","Forensic_identification_using_skin_bacterial_communities","M2Midl217_V2","ENVO:sebum","16S rRNA","ENVO:human-associated habitat","TCAG","UBERON:sebum","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","finger_tip"],"M2Ukey217.141085": ["M2Ukey217.141085","ACAGCTAGCTTG","CATGCTGCCTCCCGTAGGAGT","M2Ukey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Ukey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M2Ukey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Ukey"],"M2Zkey217.141015": ["M2Zkey217.141015","ACGACGTCTTAG","CATGCTGCCTCCCGTAGGAGT","M2Zkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Zkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M2Zkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Zkey"],"M2Kkey217.141041": ["M2Kkey217.141041","ACCTCGATCAGA","CATGCTGCCTCCCGTAGGAGT","M2Kkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Kkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M2Kkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Kkey"],"M2Enter217.141012": ["M2Enter217.141012","ACGGTGAGTGTC","CATGCTGCCTCCCGTAGGAGT","M2Enter217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Enter217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M2Enter217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Ente"],"M2Jkey217.141038": ["M2Jkey217.141038","ACCGCAGAGTCA","CATGCTGCCTCCCGTAGGAGT","M2Jkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Jkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M2Jkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Jkey"],"M2Midr217.141082": ["M2Midr217.141082","ACACACTATGGC","CATGCTGCCTCCCGTAGGAGT","M2Midr217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","human skin metagenome","0","539655","unassigned","UBERON:skin of finger","1624","CCME","7/15/08","0","FFCKVMW","ENVO:human-associated habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Midr217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","UBERON:skin","Forensic_identification_using_skin_bacterial_communities","M2Midr217_V2","ENVO:sebum","16S rRNA","ENVO:human-associated habitat","TCAG","UBERON:sebum","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","finger_tip"],"M2Ekey217.141011": ["M2Ekey217.141011","ACACTGTTCATG","CATGCTGCCTCCCGTAGGAGT","M2Ekey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Ekey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M2Ekey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Ekey"],"M2Hkey217.141026": ["M2Hkey217.141026","ACCAGCGACTAG","CATGCTGCCTCCCGTAGGAGT","M2Hkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Hkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M2Hkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Hkey"],"M2Okey217.141061": ["M2Okey217.141061","ACAGTTGCGCGA","CATGCTGCCTCCCGTAGGAGT","M2Okey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Okey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M2Okey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Okey"],"M2Bkey217.141063": ["M2Bkey217.141063","ACGCGATACTGG","CATGCTGCCTCCCGTAGGAGT","M2Bkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Bkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M2Bkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Bkey"],"M2Skey217.141089": ["M2Skey217.141089","ACATGTCACGTG","CATGCTGCCTCCCGTAGGAGT","M2Skey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Skey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M2Skey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Skey"],"M2Ckey217.141092": ["M2Ckey217.141092","ACGATGCGACCA","CATGCTGCCTCCCGTAGGAGT","M2Ckey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Ckey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M2Ckey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Ckey"],"M2Indr217.141062": ["M2Indr217.141062","AATCGTGACTCG","CATGCTGCCTCCCGTAGGAGT","M2Indr217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","human skin metagenome","0","539655","unassigned","UBERON:skin of finger","1624","CCME","7/15/08","0","FFCKVMW","ENVO:human-associated habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Indr217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","UBERON:skin","Forensic_identification_using_skin_bacterial_communities","M2Indr217_V2","ENVO:sebum","16S rRNA","ENVO:human-associated habitat","TCAG","UBERON:sebum","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","finger_tip"],"M2Gkey217.141106": ["M2Gkey217.141106","ACCAGACGATGC","CATGCTGCCTCCCGTAGGAGT","M2Gkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Gkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M2Gkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Gkey"],"M2Space217.140997": ["M2Space217.140997","ACGTACTCAGTG","CATGCTGCCTCCCGTAGGAGT","M2Space217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Space217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M2Space217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Space_bar"],"M2Lsft217.141056": ["M2Lsft217.141056","ACGGATCGTCAG","CATGCTGCCTCCCGTAGGAGT","M2Lsft217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Lsft217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M2Lsft217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Left_shift"],"M2Pkey217.141105": ["M2Pkey217.141105","ACATCACTTAGC","CATGCTGCCTCCCGTAGGAGT","M2Pkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Pkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M2Pkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Pkey"],"M2Tkey217.141054": ["M2Tkey217.141054","ACAGAGTCGGCT","CATGCTGCCTCCCGTAGGAGT","M2Tkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Tkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M2Tkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Tkey"],"M2Pinl217.141003": ["M2Pinl217.141003","AAGCTGCAGTCG","CATGCTGCCTCCCGTAGGAGT","M2Pinl217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","human skin metagenome","0","539655","unassigned","UBERON:skin of finger","1624","CCME","7/15/08","0","FFCKVMW","ENVO:human-associated habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Pinl217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","UBERON:skin","Forensic_identification_using_skin_bacterial_communities","M2Pinl217_V2","ENVO:sebum","16S rRNA","ENVO:human-associated habitat","TCAG","UBERON:sebum","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","finger_tip"],"M2Vkey217.141022": ["M2Vkey217.141022","ACGCAACTGCTA","CATGCTGCCTCCCGTAGGAGT","M2Vkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Vkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M2Vkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Vkey"],"M2Rinl217.141058": ["M2Rinl217.141058","AAGAGATGTCGA","CATGCTGCCTCCCGTAGGAGT","M2Rinl217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","human skin metagenome","0","539655","unassigned","UBERON:skin of finger","1624","CCME","7/15/08","0","FFCKVMW","ENVO:human-associated habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Rinl217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","UBERON:skin","Forensic_identification_using_skin_bacterial_communities","M2Rinl217_V2","ENVO:sebum","16S rRNA","ENVO:human-associated habitat","TCAG","UBERON:sebum","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","finger_tip"],"M2Indl217.141074": ["M2Indl217.141074","AACTCGTCGATG","CATGCTGCCTCCCGTAGGAGT","M2Indl217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","human skin metagenome","0","539655","unassigned","UBERON:skin of finger","1624","CCME","7/15/08","0","FFCKVMW","ENVO:human-associated habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Indl217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","UBERON:skin","Forensic_identification_using_skin_bacterial_communities","M2Indl217_V2","ENVO:sebum","16S rRNA","ENVO:human-associated habitat","TCAG","UBERON:sebum","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","finger_tip"],"M2Rkey217.141050": ["M2Rkey217.141050","ACAGACCACTCA","CATGCTGCCTCCCGTAGGAGT","M2Rkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Rkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M2Rkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Rkey"],"M2Fkey217.141078": ["M2Fkey217.141078","ACCACATACATC","CATGCTGCCTCCCGTAGGAGT","M2Fkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Fkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M2Fkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Fkey"],"M2Thml217.141005": ["M2Thml217.141005","AACGCACGCTAG","CATGCTGCCTCCCGTAGGAGT","M2Thml217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","human skin metagenome","0","539655","unassigned","UBERON:skin of finger","1624","CCME","7/15/08","0","FFCKVMW","ENVO:human-associated habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Thml217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","UBERON:skin","Forensic_identification_using_skin_bacterial_communities","M2Thml217_V2","ENVO:sebum","16S rRNA","ENVO:human-associated habitat","TCAG","UBERON:sebum","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","finger_tip"],"M2Dkey217.140994": ["M2Dkey217.140994","ACATTCAGCGCA","CATGCTGCCTCCCGTAGGAGT","M2Dkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Dkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M2Dkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Dkey"],"M2Ykey217.141095": ["M2Ykey217.141095","ACAGCAGTGGTC","CATGCTGCCTCCCGTAGGAGT","M2Ykey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Ykey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M2Ykey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Ykey"],"M2Ikey217.141000": ["M2Ikey217.141000","ACAGTGCTTCAT","CATGCTGCCTCCCGTAGGAGT","M2Ikey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Ikey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M2Ikey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Ikey"],"M2Wkey217.141070": ["M2Wkey217.141070","ACACTAGATCCG","CATGCTGCCTCCCGTAGGAGT","M2Wkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Wkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M2Wkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Wkey"],"M2Rsft217.141034": ["M2Rsft217.141034","ACGCTCATGGAT","CATGCTGCCTCCCGTAGGAGT","M2Rsft217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Rsft217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M2Rsft217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Right_shift"],"M2Lkey217.141048": ["M2Lkey217.141048","ACCTGTCTCTCT","CATGCTGCCTCCCGTAGGAGT","M2Lkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Lkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M2Lkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Lkey"],"M2Qkey217.141102": ["M2Qkey217.141102","ACACGGTGTCTA","CATGCTGCCTCCCGTAGGAGT","M2Qkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Qkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M2Qkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Qkey"],"M2Pinr217.141039": ["M2Pinr217.141039","ACACGAGCCACA","CATGCTGCCTCCCGTAGGAGT","M2Pinr217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","human skin metagenome","0","539655","unassigned","UBERON:skin of finger","1624","CCME","7/15/08","0","FFCKVMW","ENVO:human-associated habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Pinr217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","UBERON:skin","Forensic_identification_using_skin_bacterial_communities","M2Pinr217_V2","ENVO:sebum","16S rRNA","ENVO:human-associated habitat","TCAG","UBERON:sebum","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","finger_tip"],"M2Xkey217.141016": ["M2Xkey217.141016","ACGAGTGCTATC","CATGCTGCCTCCCGTAGGAGT","M2Xkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","36","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M2","Yes","M2Xkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M2Xkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","36","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Xkey"],"M3Rinr217.141091": ["M3Rinr217.141091","AGGTGTGATCGC","CATGCTGCCTCCCGTAGGAGT","M3Rinr217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","33","human skin metagenome","0","539655","unassigned","UBERON:skin of finger","1624","CCME","7/15/08","0","FFCKVMW","ENVO:human-associated habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M3","Yes","M3Rinr217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","UBERON:skin","Forensic_identification_using_skin_bacterial_communities","M3Rinr217_V2","ENVO:sebum","16S rRNA","ENVO:human-associated habitat","TCAG","UBERON:sebum","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","33","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","finger_tip"],"M3Vkey217.141028": ["M3Vkey217.141028","ATATCGCTACTG","CATGCTGCCTCCCGTAGGAGT","M3Vkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","33","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M3","Yes","M3Vkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M3Vkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","33","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Vkey"],"M3Hkey217.141093": ["M3Hkey217.141093","ATACACGTGGCG","CATGCTGCCTCCCGTAGGAGT","M3Hkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","33","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M3","Yes","M3Hkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M3Hkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","33","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Hkey"],"M3Rkey217.141025": ["M3Rkey217.141025","AGTCCATAGCTG","CATGCTGCCTCCCGTAGGAGT","M3Rkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","33","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M3","Yes","M3Rkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M3Rkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","33","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Rkey"],"M3Midl217.141103": ["M3Midl217.141103","AGCTCCATACAG","CATGCTGCCTCCCGTAGGAGT","M3Midl217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","33","human skin metagenome","0","539655","unassigned","UBERON:skin of finger","1624","CCME","7/15/08","0","FFCKVMW","ENVO:human-associated habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M3","Yes","M3Midl217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","UBERON:skin","Forensic_identification_using_skin_bacterial_communities","M3Midl217_V2","ENVO:sebum","16S rRNA","ENVO:human-associated habitat","TCAG","UBERON:sebum","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","33","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","finger_tip"],"M3Tkey217.141081": ["M3Tkey217.141081","AGTCTACTCTGA","CATGCTGCCTCCCGTAGGAGT","M3Tkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","33","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M3","Yes","M3Tkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M3Tkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","33","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Tkey"],"M3Wkey217.141104": ["M3Wkey217.141104","AGTAGTATCCTC","CATGCTGCCTCCCGTAGGAGT","M3Wkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","33","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M3","Yes","M3Wkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M3Wkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","33","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Wkey"],"M3Jkey217.141017": ["M3Jkey217.141017","ATACAGAGCTCC","CATGCTGCCTCCCGTAGGAGT","M3Jkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","33","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M3","Yes","M3Jkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M3Jkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","33","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Jkey"],"M3Indr217.140993": ["M3Indr217.140993","AGGACGCACTGT","CATGCTGCCTCCCGTAGGAGT","M3Indr217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","33","human skin metagenome","0","539655","unassigned","UBERON:skin of finger","1624","CCME","7/15/08","0","FFCKVMW","ENVO:human-associated habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M3","Yes","M3Indr217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","UBERON:skin","Forensic_identification_using_skin_bacterial_communities","M3Indr217_V2","ENVO:sebum","16S rRNA","ENVO:human-associated habitat","TCAG","UBERON:sebum","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","33","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","finger_tip"],"M3Ekey217.141042": ["M3Ekey217.141042","AGTCACATCACT","CATGCTGCCTCCCGTAGGAGT","M3Ekey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","33","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M3","Yes","M3Ekey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M3Ekey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","33","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Ekey"],"M3Nkey217.141001": ["M3Nkey217.141001","ATCACGTAGCGG","CATGCTGCCTCCCGTAGGAGT","M3Nkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","33","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M3","Yes","M3Nkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M3Nkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","33","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Nkey"],"M3Indl217.141067": ["M3Indl217.141067","AGCTATCCACGA","CATGCTGCCTCCCGTAGGAGT","M3Indl217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","33","human skin metagenome","0","539655","unassigned","UBERON:skin of finger","1624","CCME","7/15/08","0","FFCKVMW","ENVO:human-associated habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M3","Yes","M3Indl217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","UBERON:skin","Forensic_identification_using_skin_bacterial_communities","M3Indl217_V2","ENVO:sebum","16S rRNA","ENVO:human-associated habitat","TCAG","UBERON:sebum","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","33","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","finger_tip"],"M3Space217.141098": ["M3Space217.141098","ATCGATCTGTGG","CATGCTGCCTCCCGTAGGAGT","M3Space217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","33","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M3","Yes","M3Space217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M3Space217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","33","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Space_bar"],"M3Gkey217.141037": ["M3Gkey217.141037","ATAATCTCGTCG","CATGCTGCCTCCCGTAGGAGT","M3Gkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","33","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M3","Yes","M3Gkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M3Gkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","33","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Gkey"],"M3Mkey217.141107": ["M3Mkey217.141107","ATCACTAGTCAC","CATGCTGCCTCCCGTAGGAGT","M3Mkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","33","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M3","Yes","M3Mkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M3Mkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","33","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Mkey"],"M3Ykey217.141023": ["M3Ykey217.141023","AGTCTCGCATAT","CATGCTGCCTCCCGTAGGAGT","M3Ykey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","33","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M3","Yes","M3Ykey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M3Ykey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","33","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Ykey"],"M3Pinl217.141094": ["M3Pinl217.141094","AGCTGACTAGTC","CATGCTGCCTCCCGTAGGAGT","M3Pinl217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","33","human skin metagenome","0","539655","unassigned","UBERON:skin of finger","1624","CCME","7/15/08","0","FFCKVMW","ENVO:human-associated habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M3","Yes","M3Pinl217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","UBERON:skin","Forensic_identification_using_skin_bacterial_communities","M3Pinl217_V2","ENVO:sebum","16S rRNA","ENVO:human-associated habitat","TCAG","UBERON:sebum","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","33","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","finger_tip"],"M3Midr217.141101": ["M3Midr217.141101","AGGCTACACGAC","CATGCTGCCTCCCGTAGGAGT","M3Midr217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","33","human skin metagenome","0","539655","unassigned","UBERON:skin of finger","1624","CCME","7/15/08","0","FFCKVMW","ENVO:human-associated habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M3","Yes","M3Midr217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","UBERON:skin","Forensic_identification_using_skin_bacterial_communities","M3Midr217_V2","ENVO:sebum","16S rRNA","ENVO:human-associated habitat","TCAG","UBERON:sebum","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","33","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","finger_tip"],"M3Rsft217.140995": ["M3Rsft217.140995","ATCCGATCACAG","CATGCTGCCTCCCGTAGGAGT","M3Rsft217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","33","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M3","Yes","M3Rsft217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M3Rsft217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","33","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Right_shift"],"M3Thml217.141040": ["M3Thml217.141040","AGCGTAGGTCGT","CATGCTGCCTCCCGTAGGAGT","M3Thml217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","33","human skin metagenome","0","539655","unassigned","UBERON:skin of finger","1624","CCME","7/15/08","0","FFCKVMW","ENVO:human-associated habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M3","Yes","M3Thml217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","UBERON:skin","Forensic_identification_using_skin_bacterial_communities","M3Thml217_V2","ENVO:sebum","16S rRNA","ENVO:human-associated habitat","TCAG","UBERON:sebum","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","33","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","finger_tip"],"M3Xkey217.141055": ["M3Xkey217.141055","ATAGCTCCATAC","CATGCTGCCTCCCGTAGGAGT","M3Xkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","33","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M3","Yes","M3Xkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M3Xkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","33","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Xkey"],"M3Ckey217.141073": ["M3Ckey217.141073","ATAGGCGATCTC","CATGCTGCCTCCCGTAGGAGT","M3Ckey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","33","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M3","Yes","M3Ckey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M3Ckey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","33","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Ckey"],"M3Qkey217.141051": ["M3Qkey217.141051","AGTACTGCAGGC","CATGCTGCCTCCCGTAGGAGT","M3Qkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","33","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M3","Yes","M3Qkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M3Qkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","33","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Qkey"],"M3Pkey217.141031": ["M3Pkey217.141031","AGTGTCACGGTG","CATGCTGCCTCCCGTAGGAGT","M3Pkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","33","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M3","Yes","M3Pkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M3Pkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","33","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Pkey"],"M3Rinl217.141100": ["M3Rinl217.141100","AGCTCTCAGAGG","CATGCTGCCTCCCGTAGGAGT","M3Rinl217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","33","human skin metagenome","0","539655","unassigned","UBERON:skin of finger","1624","CCME","7/15/08","0","FFCKVMW","ENVO:human-associated habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M3","Yes","M3Rinl217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","UBERON:skin","Forensic_identification_using_skin_bacterial_communities","M3Rinl217_V2","ENVO:sebum","16S rRNA","ENVO:human-associated habitat","TCAG","UBERON:sebum","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","33","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","finger_tip"],"M3Lkey217.141097": ["M3Lkey217.141097","ATACTATTGCGC","CATGCTGCCTCCCGTAGGAGT","M3Lkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","33","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M3","Yes","M3Lkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M3Lkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","33","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Lkey"],"M3Thmr217.140996": ["M3Thmr217.140996","AGCTTGACAGCT","CATGCTGCCTCCCGTAGGAGT","M3Thmr217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","33","human skin metagenome","0","539655","unassigned","UBERON:skin of finger","1624","CCME","7/15/08","0","FFCKVMW","ENVO:human-associated habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M3","Yes","M3Thmr217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","UBERON:skin","Forensic_identification_using_skin_bacterial_communities","M3Thmr217_V2","ENVO:sebum","16S rRNA","ENVO:human-associated habitat","TCAG","UBERON:sebum","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","33","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","finger_tip"],"M3Bkey217.141059": ["M3Bkey217.141059","ATATGCCAGTGC","CATGCTGCCTCCCGTAGGAGT","M3Bkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","33","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M3","Yes","M3Bkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M3Bkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","33","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Bkey"],"M3Akey217.141006": ["M3Akey217.141006","AGTGTTCGATCG","CATGCTGCCTCCCGTAGGAGT","M3Akey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","33","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M3","Yes","M3Akey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M3Akey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","33","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Akey"],"M3Lsft217.141088": ["M3Lsft217.141088","ATCAGGCGTGTG","CATGCTGCCTCCCGTAGGAGT","M3Lsft217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","33","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M3","Yes","M3Lsft217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M3Lsft217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","33","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Left_Shift"],"M3Kkey217.141046": ["M3Kkey217.141046","ATACGTCTTCGA","CATGCTGCCTCCCGTAGGAGT","M3Kkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","33","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M3","Yes","M3Kkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M3Kkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","33","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Kkey"],"M3Pinr217.141057": ["M3Pinr217.141057","AGTACGCTCGAG","CATGCTGCCTCCCGTAGGAGT","M3Pinr217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","33","human skin metagenome","0","539655","unassigned","UBERON:skin of finger","1624","CCME","7/15/08","0","FFCKVMW","ENVO:human-associated habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M3","Yes","M3Pinr217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","UBERON:skin","Forensic_identification_using_skin_bacterial_communities","M3Pinr217_V2","ENVO:sebum","16S rRNA","ENVO:human-associated habitat","TCAG","UBERON:sebum","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","33","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","finger_tip"],"M3Zkey217.141083": ["M3Zkey217.141083","ATACTCACTCAG","CATGCTGCCTCCCGTAGGAGT","M3Zkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","33","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M3","Yes","M3Zkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M3Zkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","33","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Zkey"],"M9Pinl217.141035": ["M9Pinl217.141035","ACTACAGCCTAT","CATGCTGCCTCCCGTAGGAGT","M9Pinl217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","25","human skin metagenome","0","539655","unassigned","UBERON:skin of finger","1624","CCME","7/15/08","0","FFCKVMW","ENVO:human-associated habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M9","Yes","M9Pinl217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","UBERON:skin","Forensic_identification_using_skin_bacterial_communities","M9Pinl217_V2","ENVO:sebum","16S rRNA","ENVO:human-associated habitat","TCAG","UBERON:sebum","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","25","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","finger_tip"],"M9Enter217.141084": ["M9Enter217.141084","AGCGAGCTATCT","CATGCTGCCTCCCGTAGGAGT","M9Enter217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","25","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M9","Yes","M9Enter217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M9Enter217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","25","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Ente"],"M9Thmr217.141049": ["M9Thmr217.141049","ACTACGTGTGGT","CATGCTGCCTCCCGTAGGAGT","M9Thmr217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","25","human skin metagenome","0","539655","unassigned","UBERON:skin of finger","1624","CCME","7/15/08","0","FFCKVMW","ENVO:human-associated habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M9","Yes","M9Thmr217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","UBERON:skin","Forensic_identification_using_skin_bacterial_communities","M9Thmr217_V2","ENVO:sebum","16S rRNA","ENVO:human-associated habitat","TCAG","UBERON:sebum","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","25","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","finger_tip"],"M9Thml217.141021": ["M9Thml217.141021","ACGTCTGTAGCA","CATGCTGCCTCCCGTAGGAGT","M9Thml217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","25","human skin metagenome","0","539655","unassigned","UBERON:skin of finger","1624","CCME","7/15/08","0","FFCKVMW","ENVO:human-associated habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M9","Yes","M9Thml217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","UBERON:skin","Forensic_identification_using_skin_bacterial_communities","M9Thml217_V2","ENVO:sebum","16S rRNA","ENVO:human-associated habitat","TCAG","UBERON:sebum","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","25","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","finger_tip"],"M9Midr217.141060": ["M9Midr217.141060","ACTATTGTCACG","CATGCTGCCTCCCGTAGGAGT","M9Midr217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","25","human skin metagenome","0","539655","unassigned","UBERON:skin of finger","1624","CCME","7/15/08","0","FFCKVMW","ENVO:human-associated habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M9","Yes","M9Midr217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","UBERON:skin","Forensic_identification_using_skin_bacterial_communities","M9Midr217_V2","ENVO:sebum","16S rRNA","ENVO:human-associated habitat","TCAG","UBERON:sebum","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","25","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","finger_tip"],"M9Qkey217.141019": ["M9Qkey217.141019","ACTCGATTCGAT","CATGCTGCCTCCCGTAGGAGT","M9Qkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","25","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M9","Yes","M9Qkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M9Qkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","25","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Qkey"],"M9Indl217.141066": ["M9Indl217.141066","ACGTGAGAGAAT","CATGCTGCCTCCCGTAGGAGT","M9Indl217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","25","human skin metagenome","0","539655","unassigned","UBERON:skin of finger","1624","CCME","7/15/08","0","FFCKVMW","ENVO:human-associated habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M9","Yes","M9Indl217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","UBERON:skin","Forensic_identification_using_skin_bacterial_communities","M9Indl217_V2","ENVO:sebum","16S rRNA","ENVO:human-associated habitat","TCAG","UBERON:sebum","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","25","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","finger_tip"],"M9Rinl217.141020": ["M9Rinl217.141020","ACGTTAGCACAC","CATGCTGCCTCCCGTAGGAGT","M9Rinl217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","25","human skin metagenome","0","539655","unassigned","UBERON:skin of finger","1624","CCME","7/15/08","0","FFCKVMW","ENVO:human-associated habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M9","Yes","M9Rinl217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","UBERON:skin","Forensic_identification_using_skin_bacterial_communities","M9Rinl217_V2","ENVO:sebum","16S rRNA","ENVO:human-associated habitat","TCAG","UBERON:sebum","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","25","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","finger_tip"],"M9Gkey217.141099": ["M9Gkey217.141099","AGAGCAAGAGCA","CATGCTGCCTCCCGTAGGAGT","M9Gkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","25","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M9","Yes","M9Gkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M9Gkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","25","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Gkey"],"M9Ckey217.141013": ["M9Ckey217.141013","AGCACACCTACA","CATGCTGCCTCCCGTAGGAGT","M9Ckey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","25","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M9","Yes","M9Ckey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M9Ckey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","25","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Ckey"],"M9Okey217.141076": ["M9Okey217.141076","ACTTGTAGCAGC","CATGCTGCCTCCCGTAGGAGT","M9Okey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","25","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M9","Yes","M9Okey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M9Okey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","25","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Okey"],"M9Midl217.141043": ["M9Midl217.141043","ACGTGCCGTAGA","CATGCTGCCTCCCGTAGGAGT","M9Midl217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","25","human skin metagenome","0","539655","unassigned","UBERON:skin of finger","1624","CCME","7/15/08","0","FFCKVMW","ENVO:human-associated habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M9","Yes","M9Midl217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","UBERON:skin","Forensic_identification_using_skin_bacterial_communities","M9Midl217_V2","ENVO:sebum","16S rRNA","ENVO:human-associated habitat","TCAG","UBERON:sebum","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","25","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","finger_tip"],"M9Space217.141079": ["M9Space217.141079","AGCGCTGATGTG","CATGCTGCCTCCCGTAGGAGT","M9Space217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","25","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M9","Yes","M9Space217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M9Space217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","25","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Space_bar"],"M9Hkey217.140999": ["M9Hkey217.140999","AGAGTAGCTAAG","CATGCTGCCTCCCGTAGGAGT","M9Hkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","25","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M9","Yes","M9Hkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M9Hkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","25","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Hkey"],"M9Ykey217.141029": ["M9Ykey217.141029","ACTGTACGCGTA","CATGCTGCCTCCCGTAGGAGT","M9Ykey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","25","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M9","Yes","M9Ykey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M9Ykey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","25","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Ykey"],"M9Akey217.141086": ["M9Akey217.141086","AGACCGTCAGAC","CATGCTGCCTCCCGTAGGAGT","M9Akey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","25","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M9","Yes","M9Akey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M9Akey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","25","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Akey"],"M9Mkey217.141007": ["M9Mkey217.141007","AGCATATGAGAG","CATGCTGCCTCCCGTAGGAGT","M9Mkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","25","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M9","Yes","M9Mkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M9Mkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","25","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Mkey"],"M9Xkey217.141072": ["M9Xkey217.141072","AGATGTTCTGCT","CATGCTGCCTCCCGTAGGAGT","M9Xkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","25","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M9","Yes","M9Xkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M9Xkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","25","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Xkey"],"M9Ekey217.141044": ["M9Ekey217.141044","ACTCTTCTAGAG","CATGCTGCCTCCCGTAGGAGT","M9Ekey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","25","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M9","Yes","M9Ekey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M9Ekey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","25","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Ekey"],"M9Indr217.140998": ["M9Indr217.140998","ACTAGCTCCATA","CATGCTGCCTCCCGTAGGAGT","M9Indr217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","25","human skin metagenome","0","539655","unassigned","UBERON:skin of finger","1624","CCME","7/15/08","0","FFCKVMW","ENVO:human-associated habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M9","Yes","M9Indr217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","UBERON:skin","Forensic_identification_using_skin_bacterial_communities","M9Indr217_V2","ENVO:sebum","16S rRNA","ENVO:human-associated habitat","TCAG","UBERON:sebum","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","25","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","finger_tip"],"M9Vkey217.141018": ["M9Vkey217.141018","AGCACGAGCCTA","CATGCTGCCTCCCGTAGGAGT","M9Vkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","25","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M9","Yes","M9Vkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M9Vkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","25","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Vkey"],"M9Dkey217.141053": ["M9Dkey217.141053","AGACTGCGTACT","CATGCTGCCTCCCGTAGGAGT","M9Dkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","25","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M9","Yes","M9Dkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M9Dkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","25","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Dkey"],"M9Bkey217.141032": ["M9Bkey217.141032","AGCAGCACTTGT","CATGCTGCCTCCCGTAGGAGT","M9Bkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","25","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M9","Yes","M9Bkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M9Bkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","25","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Bkey"],"M9Rinr217.141080": ["M9Rinr217.141080","ACTCACGGTATG","CATGCTGCCTCCCGTAGGAGT","M9Rinr217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","25","human skin metagenome","0","539655","unassigned","UBERON:skin of finger","1624","CCME","7/15/08","0","FFCKVMW","ENVO:human-associated habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M9","Yes","M9Rinr217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","UBERON:skin","Forensic_identification_using_skin_bacterial_communities","M9Rinr217_V2","ENVO:sebum","16S rRNA","ENVO:human-associated habitat","TCAG","UBERON:sebum","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","25","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","finger_tip"],"M9Kkey217.141033": ["M9Kkey217.141033","AGATACACGCGC","CATGCTGCCTCCCGTAGGAGT","M9Kkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","25","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M9","Yes","M9Kkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M9Kkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","25","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Kkey"],"M9Skey217.141004": ["M9Skey217.141004","AGACGTGCACTG","CATGCTGCCTCCCGTAGGAGT","M9Skey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","25","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M9","Yes","M9Skey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M9Skey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","25","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Skey"],"M9Pkey217.141096": ["M9Pkey217.141096","AGAACACGTCTC","CATGCTGCCTCCCGTAGGAGT","M9Pkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","25","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M9","Yes","M9Pkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M9Pkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","25","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Pkey"],"M9Nkey217.141010": ["M9Nkey217.141010","AGCAGTCGCGAT","CATGCTGCCTCCCGTAGGAGT","M9Nkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","25","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M9","Yes","M9Nkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M9Nkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","25","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Nkey"],"M9Fkey217.141065": ["M9Fkey217.141065","AGAGAGCAAGTG","CATGCTGCCTCCCGTAGGAGT","M9Fkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","25","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M9","Yes","M9Fkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M9Fkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","25","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Fkey"],"M9Wkey217.141090": ["M9Wkey217.141090","ACTCGCACAGGA","CATGCTGCCTCCCGTAGGAGT","M9Wkey217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","25","computer keyboard metagenome","0","256318","unassigned","NA","1624","CCME","7/15/08","0","FFCKVMW","ENVO:anthropogenic habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M9","Yes","M9Wkey217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","NA","Forensic_identification_using_skin_bacterial_communities","M9Wkey217_V2","ENVO:surface","16S rRNA","ENVO:surface","TCAG","NA","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","25","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","Wkey"],"M9Pinr217.141002": ["M9Pinr217.141002","ACTCAGATACTC","CATGCTGCCTCCCGTAGGAGT","M9Pinr217_V2","V2","n","8/14/08","Forensic_identification_using_skin_bacterial_communities","25","human skin metagenome","0","539655","unassigned","UBERON:skin of finger","1624","CCME","7/15/08","0","FFCKVMW","ENVO:human-associated habitat","male","FLX","Forensic_identification_using_skin_bacterial_communities","NA","M9","Yes","M9Pinr217","0.8","CCME","1, swab","-105.2705","232","fierer_forensic_keyboard","years","UBERON:skin","Forensic_identification_using_skin_bacterial_communities","M9Pinr217_V2","ENVO:sebum","16S rRNA","ENVO:human-associated habitat","TCAG","UBERON:sebum","GAZ:United States of America","16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_<200_or_>300_nt_or_with_average_quality_scores_of_<25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12_nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_&_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150_nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_&_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_&_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).","0","CCME","V2","FWD:GCCTTGCCAGCCCGCTCAGTCAGAGTTTGATCCTGGCTCAG;REV:TGCTGCCTCCCGTAGGAGT","25","pyrosequencing","NA","Forensic_identification_using_skin_bacterial_communities","40.0083","Forensic_identification_using_skin_bacterial_communities","y","CCME","finger_tip"]}
var ellipses = new Array()
var points = new Array()
ellipses['M3Gkey217.141037'] = { 'name': 'M3Gkey217.141037', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.158536, 'y': 0.136268, 'z': 0.006955 }
points['M3Gkey217.141037'] = { 'name': 'M3Gkey217.141037', 'color': 0, 'x': -0.158536, 'y': 0.136268, 'z': 0.006955 }
ellipses['M9Gkey217.141099'] = { 'name': 'M9Gkey217.141099', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.133001, 'y': -0.119027, 'z': 0.034362 }
points['M9Gkey217.141099'] = { 'name': 'M9Gkey217.141099', 'color': 0, 'x': -0.133001, 'y': -0.119027, 'z': 0.034362 }
ellipses['M3Ekey217.141042'] = { 'name': 'M3Ekey217.141042', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.165373, 'y': 0.123989, 'z': 0.054741 }
points['M3Ekey217.141042'] = { 'name': 'M3Ekey217.141042', 'color': 0, 'x': -0.165373, 'y': 0.123989, 'z': 0.054741 }
ellipses['M3Nkey217.141001'] = { 'name': 'M3Nkey217.141001', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.228488, 'y': 0.042617, 'z': 0.146505 }
points['M3Nkey217.141001'] = { 'name': 'M3Nkey217.141001', 'color': 0, 'x': -0.228488, 'y': 0.042617, 'z': 0.146505 }
ellipses['M9Ykey217.141029'] = { 'name': 'M9Ykey217.141029', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.262587, 'y': -0.102591, 'z': 0.071217 }
points['M9Ykey217.141029'] = { 'name': 'M9Ykey217.141029', 'color': 0, 'x': -0.262587, 'y': -0.102591, 'z': 0.071217 }
ellipses['M9Nkey217.141010'] = { 'name': 'M9Nkey217.141010', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.245714, 'y': -0.104298, 'z': 0.060222 }
points['M9Nkey217.141010'] = { 'name': 'M9Nkey217.141010', 'color': 0, 'x': -0.245714, 'y': -0.104298, 'z': 0.060222 }
ellipses['M3Ykey217.141023'] = { 'name': 'M3Ykey217.141023', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.118770, 'y': 0.169166, 'z': -0.000468 }
points['M3Ykey217.141023'] = { 'name': 'M3Ykey217.141023', 'color': 0, 'x': -0.118770, 'y': 0.169166, 'z': -0.000468 }
ellipses['M2Lkey217.141048'] = { 'name': 'M2Lkey217.141048', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.200404, 'y': 0.022873, 'z': 0.024599 }
points['M2Lkey217.141048'] = { 'name': 'M2Lkey217.141048', 'color': 0, 'x': 0.200404, 'y': 0.022873, 'z': 0.024599 }
ellipses['M9Indl217.141066'] = { 'name': 'M9Indl217.141066', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.046732, 'y': -0.102359, 'z': -0.218662 }
points['M9Indl217.141066'] = { 'name': 'M9Indl217.141066', 'color': 0, 'x': -0.046732, 'y': -0.102359, 'z': -0.218662 }
ellipses['M9Hkey217.140999'] = { 'name': 'M9Hkey217.140999', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.190227, 'y': -0.197864, 'z': 0.027293 }
points['M9Hkey217.140999'] = { 'name': 'M9Hkey217.140999', 'color': 0, 'x': -0.190227, 'y': -0.197864, 'z': 0.027293 }
ellipses['M3Rinl217.141100'] = { 'name': 'M3Rinl217.141100', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.172055, 'y': 0.041934, 'z': 0.004847 }
points['M3Rinl217.141100'] = { 'name': 'M3Rinl217.141100', 'color': 0, 'x': -0.172055, 'y': 0.041934, 'z': 0.004847 }
ellipses['M2Vkey217.141022'] = { 'name': 'M2Vkey217.141022', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.322457, 'y': -0.003810, 'z': -0.013606 }
points['M2Vkey217.141022'] = { 'name': 'M2Vkey217.141022', 'color': 0, 'x': 0.322457, 'y': -0.003810, 'z': -0.013606 }
ellipses['M9Kkey217.141033'] = { 'name': 'M9Kkey217.141033', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.226876, 'y': -0.251755, 'z': 0.174146 }
points['M9Kkey217.141033'] = { 'name': 'M9Kkey217.141033', 'color': 0, 'x': -0.226876, 'y': -0.251755, 'z': 0.174146 }
ellipses['M9Pinl217.141035'] = { 'name': 'M9Pinl217.141035', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.078926, 'y': 0.018160, 'z': -0.193355 }
points['M9Pinl217.141035'] = { 'name': 'M9Pinl217.141035', 'color': 0, 'x': -0.078926, 'y': 0.018160, 'z': -0.193355 }
ellipses['M2Rsft217.141034'] = { 'name': 'M2Rsft217.141034', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.303220, 'y': -0.022643, 'z': 0.015331 }
points['M2Rsft217.141034'] = { 'name': 'M2Rsft217.141034', 'color': 0, 'x': 0.303220, 'y': -0.022643, 'z': 0.015331 }
ellipses['M9Enter217.141084'] = { 'name': 'M9Enter217.141084', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.218102, 'y': -0.091683, 'z': 0.063601 }
points['M9Enter217.141084'] = { 'name': 'M9Enter217.141084', 'color': 0, 'x': -0.218102, 'y': -0.091683, 'z': 0.063601 }
ellipses['M3Rinr217.141091'] = { 'name': 'M3Rinr217.141091', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.045275, 'y': 0.173004, 'z': -0.080758 }
points['M3Rinr217.141091'] = { 'name': 'M3Rinr217.141091', 'color': 0, 'x': -0.045275, 'y': 0.173004, 'z': -0.080758 }
ellipses['M2Dkey217.140994'] = { 'name': 'M2Dkey217.140994', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.375335, 'y': -0.093183, 'z': 0.112299 }
points['M2Dkey217.140994'] = { 'name': 'M2Dkey217.140994', 'color': 0, 'x': 0.375335, 'y': -0.093183, 'z': 0.112299 }
ellipses['M9Bkey217.141032'] = { 'name': 'M9Bkey217.141032', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.241319, 'y': -0.090301, 'z': -0.056054 }
points['M9Bkey217.141032'] = { 'name': 'M9Bkey217.141032', 'color': 0, 'x': -0.241319, 'y': -0.090301, 'z': -0.056054 }
ellipses['M2Midr217.141082'] = { 'name': 'M2Midr217.141082', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.198911, 'y': 0.024279, 'z': -0.109715 }
points['M2Midr217.141082'] = { 'name': 'M2Midr217.141082', 'color': 0, 'x': 0.198911, 'y': 0.024279, 'z': -0.109715 }
ellipses['M2Ukey217.141085'] = { 'name': 'M2Ukey217.141085', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.194882, 'y': 0.027144, 'z': -0.035865 }
points['M2Ukey217.141085'] = { 'name': 'M2Ukey217.141085', 'color': 0, 'x': 0.194882, 'y': 0.027144, 'z': -0.035865 }
ellipses['M9Ekey217.141044'] = { 'name': 'M9Ekey217.141044', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.183877, 'y': -0.078836, 'z': 0.066126 }
points['M9Ekey217.141044'] = { 'name': 'M9Ekey217.141044', 'color': 0, 'x': -0.183877, 'y': -0.078836, 'z': 0.066126 }
ellipses['M3Vkey217.141028'] = { 'name': 'M3Vkey217.141028', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.191184, 'y': 0.041199, 'z': 0.046429 }
points['M3Vkey217.141028'] = { 'name': 'M3Vkey217.141028', 'color': 0, 'x': -0.191184, 'y': 0.041199, 'z': 0.046429 }
ellipses['M9Wkey217.141090'] = { 'name': 'M9Wkey217.141090', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.207664, 'y': -0.210066, 'z': 0.031394 }
points['M9Wkey217.141090'] = { 'name': 'M9Wkey217.141090', 'color': 0, 'x': -0.207664, 'y': -0.210066, 'z': 0.031394 }
ellipses['M3Indr217.140993'] = { 'name': 'M3Indr217.140993', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.081215, 'y': 0.222702, 'z': 0.018047 }
points['M3Indr217.140993'] = { 'name': 'M3Indr217.140993', 'color': 0, 'x': -0.081215, 'y': 0.222702, 'z': 0.018047 }
ellipses['M3Indl217.141067'] = { 'name': 'M3Indl217.141067', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.146940, 'y': 0.054618, 'z': -0.087757 }
points['M3Indl217.141067'] = { 'name': 'M3Indl217.141067', 'color': 0, 'x': -0.146940, 'y': 0.054618, 'z': -0.087757 }
ellipses['M2Ikey217.141000'] = { 'name': 'M2Ikey217.141000', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.260930, 'y': 0.067694, 'z': 0.022968 }
points['M2Ikey217.141000'] = { 'name': 'M2Ikey217.141000', 'color': 0, 'x': 0.260930, 'y': 0.067694, 'z': 0.022968 }
ellipses['M2Ykey217.141095'] = { 'name': 'M2Ykey217.141095', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.232304, 'y': 0.059304, 'z': -0.018013 }
points['M2Ykey217.141095'] = { 'name': 'M2Ykey217.141095', 'color': 0, 'x': 0.232304, 'y': 0.059304, 'z': -0.018013 }
ellipses['M2Mkey217.141014'] = { 'name': 'M2Mkey217.141014', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.133532, 'y': 0.013839, 'z': -0.061731 }
points['M2Mkey217.141014'] = { 'name': 'M2Mkey217.141014', 'color': 0, 'x': 0.133532, 'y': 0.013839, 'z': -0.061731 }
ellipses['M9Xkey217.141072'] = { 'name': 'M9Xkey217.141072', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.233382, 'y': -0.101035, 'z': -0.029716 }
points['M9Xkey217.141072'] = { 'name': 'M9Xkey217.141072', 'color': 0, 'x': -0.233382, 'y': -0.101035, 'z': -0.029716 }
ellipses['M3Xkey217.141055'] = { 'name': 'M3Xkey217.141055', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.108365, 'y': 0.198130, 'z': -0.050879 }
points['M3Xkey217.141055'] = { 'name': 'M3Xkey217.141055', 'color': 0, 'x': -0.108365, 'y': 0.198130, 'z': -0.050879 }
ellipses['M2Akey217.141030'] = { 'name': 'M2Akey217.141030', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.324918, 'y': -0.032707, 'z': 0.017123 }
points['M2Akey217.141030'] = { 'name': 'M2Akey217.141030', 'color': 0, 'x': 0.324918, 'y': -0.032707, 'z': 0.017123 }
ellipses['M2Hkey217.141026'] = { 'name': 'M2Hkey217.141026', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.243430, 'y': 0.002521, 'z': -0.044523 }
points['M2Hkey217.141026'] = { 'name': 'M2Hkey217.141026', 'color': 0, 'x': 0.243430, 'y': 0.002521, 'z': -0.044523 }
ellipses['M9Qkey217.141019'] = { 'name': 'M9Qkey217.141019', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.235538, 'y': -0.162245, 'z': 0.068697 }
points['M9Qkey217.141019'] = { 'name': 'M9Qkey217.141019', 'color': 0, 'x': -0.235538, 'y': -0.162245, 'z': 0.068697 }
ellipses['M9Thmr217.141049'] = { 'name': 'M9Thmr217.141049', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.071572, 'y': -0.031029, 'z': -0.259952 }
points['M9Thmr217.141049'] = { 'name': 'M9Thmr217.141049', 'color': 0, 'x': -0.071572, 'y': -0.031029, 'z': -0.259952 }
ellipses['M3Midr217.141101'] = { 'name': 'M3Midr217.141101', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.031541, 'y': 0.229626, 'z': -0.042358 }
points['M3Midr217.141101'] = { 'name': 'M3Midr217.141101', 'color': 0, 'x': -0.031541, 'y': 0.229626, 'z': -0.042358 }
ellipses['M2Enter217.141012'] = { 'name': 'M2Enter217.141012', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.281288, 'y': 0.027440, 'z': -0.031682 }
points['M2Enter217.141012'] = { 'name': 'M2Enter217.141012', 'color': 0, 'x': 0.281288, 'y': 0.027440, 'z': -0.031682 }
ellipses['M3Space217.141098'] = { 'name': 'M3Space217.141098', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.211787, 'y': 0.170133, 'z': 0.144692 }
points['M3Space217.141098'] = { 'name': 'M3Space217.141098', 'color': 0, 'x': -0.211787, 'y': 0.170133, 'z': 0.144692 }
ellipses['M2Okey217.141061'] = { 'name': 'M2Okey217.141061', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.193423, 'y': 0.016295, 'z': -0.015464 }
points['M2Okey217.141061'] = { 'name': 'M2Okey217.141061', 'color': 0, 'x': 0.193423, 'y': 0.016295, 'z': -0.015464 }
ellipses['M9Rinl217.141020'] = { 'name': 'M9Rinl217.141020', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.145455, 'y': -0.019749, 'z': -0.190103 }
points['M9Rinl217.141020'] = { 'name': 'M9Rinl217.141020', 'color': 0, 'x': -0.145455, 'y': -0.019749, 'z': -0.190103 }
ellipses['M2Lsft217.141056'] = { 'name': 'M2Lsft217.141056', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.241483, 'y': 0.006373, 'z': -0.051895 }
points['M2Lsft217.141056'] = { 'name': 'M2Lsft217.141056', 'color': 0, 'x': 0.241483, 'y': 0.006373, 'z': -0.051895 }
ellipses['M2Tkey217.141054'] = { 'name': 'M2Tkey217.141054', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.296187, 'y': -0.000372, 'z': 0.010935 }
points['M2Tkey217.141054'] = { 'name': 'M2Tkey217.141054', 'color': 0, 'x': 0.296187, 'y': -0.000372, 'z': 0.010935 }
ellipses['M3Kkey217.141046'] = { 'name': 'M3Kkey217.141046', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.203404, 'y': -0.002844, 'z': 0.163083 }
points['M3Kkey217.141046'] = { 'name': 'M3Kkey217.141046', 'color': 0, 'x': -0.203404, 'y': -0.002844, 'z': 0.163083 }
ellipses['M9Pkey217.141096'] = { 'name': 'M9Pkey217.141096', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.264642, 'y': -0.128048, 'z': 0.188123 }
points['M9Pkey217.141096'] = { 'name': 'M9Pkey217.141096', 'color': 0, 'x': -0.264642, 'y': -0.128048, 'z': 0.188123 }
ellipses['M9Midl217.141043'] = { 'name': 'M9Midl217.141043', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.165848, 'y': -0.102021, 'z': -0.163375 }
points['M9Midl217.141043'] = { 'name': 'M9Midl217.141043', 'color': 0, 'x': -0.165848, 'y': -0.102021, 'z': -0.163375 }
ellipses['M3Tkey217.141081'] = { 'name': 'M3Tkey217.141081', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.138512, 'y': 0.083006, 'z': -0.017960 }
points['M3Tkey217.141081'] = { 'name': 'M3Tkey217.141081', 'color': 0, 'x': -0.138512, 'y': 0.083006, 'z': -0.017960 }
ellipses['M2Xkey217.141016'] = { 'name': 'M2Xkey217.141016', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.261329, 'y': -0.103909, 'z': 0.126256 }
points['M2Xkey217.141016'] = { 'name': 'M2Xkey217.141016', 'color': 0, 'x': 0.261329, 'y': -0.103909, 'z': 0.126256 }
ellipses['M2Pinr217.141039'] = { 'name': 'M2Pinr217.141039', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.157742, 'y': 0.074467, 'z': -0.055804 }
points['M2Pinr217.141039'] = { 'name': 'M2Pinr217.141039', 'color': 0, 'x': 0.157742, 'y': 0.074467, 'z': -0.055804 }
ellipses['M9Fkey217.141065'] = { 'name': 'M9Fkey217.141065', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.228869, 'y': -0.126066, 'z': -0.043772 }
points['M9Fkey217.141065'] = { 'name': 'M9Fkey217.141065', 'color': 0, 'x': -0.228869, 'y': -0.126066, 'z': -0.043772 }
ellipses['M2Bkey217.141063'] = { 'name': 'M2Bkey217.141063', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.299785, 'y': -0.028365, 'z': -0.051652 }
points['M2Bkey217.141063'] = { 'name': 'M2Bkey217.141063', 'color': 0, 'x': 0.299785, 'y': -0.028365, 'z': -0.051652 }
ellipses['M9Dkey217.141053'] = { 'name': 'M9Dkey217.141053', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.242269, 'y': -0.136108, 'z': 0.007296 }
points['M9Dkey217.141053'] = { 'name': 'M9Dkey217.141053', 'color': 0, 'x': -0.242269, 'y': -0.136108, 'z': 0.007296 }
ellipses['M3Zkey217.141083'] = { 'name': 'M3Zkey217.141083', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.226585, 'y': 0.076836, 'z': 0.238494 }
points['M3Zkey217.141083'] = { 'name': 'M3Zkey217.141083', 'color': 0, 'x': -0.226585, 'y': 0.076836, 'z': 0.238494 }
ellipses['M9Rinr217.141080'] = { 'name': 'M9Rinr217.141080', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.142361, 'y': -0.100038, 'z': -0.246415 }
points['M9Rinr217.141080'] = { 'name': 'M9Rinr217.141080', 'color': 0, 'x': -0.142361, 'y': -0.100038, 'z': -0.246415 }
ellipses['M2Indl217.141074'] = { 'name': 'M2Indl217.141074', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.256162, 'y': -0.010918, 'z': -0.019466 }
points['M2Indl217.141074'] = { 'name': 'M2Indl217.141074', 'color': 0, 'x': 0.256162, 'y': -0.010918, 'z': -0.019466 }
ellipses['M9Pinr217.141002'] = { 'name': 'M9Pinr217.141002', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.118226, 'y': -0.181541, 'z': -0.104564 }
points['M9Pinr217.141002'] = { 'name': 'M9Pinr217.141002', 'color': 0, 'x': -0.118226, 'y': -0.181541, 'z': -0.104564 }
ellipses['M3Thmr217.140996'] = { 'name': 'M3Thmr217.140996', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.086480, 'y': 0.247867, 'z': 0.005684 }
points['M3Thmr217.140996'] = { 'name': 'M3Thmr217.140996', 'color': 0, 'x': -0.086480, 'y': 0.247867, 'z': 0.005684 }
ellipses['M2Rinr217.141027'] = { 'name': 'M2Rinr217.141027', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.216951, 'y': 0.001487, 'z': -0.081013 }
points['M2Rinr217.141027'] = { 'name': 'M2Rinr217.141027', 'color': 0, 'x': 0.216951, 'y': 0.001487, 'z': -0.081013 }
ellipses['M3Mkey217.141107'] = { 'name': 'M3Mkey217.141107', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.135634, 'y': 0.142330, 'z': 0.126765 }
points['M3Mkey217.141107'] = { 'name': 'M3Mkey217.141107', 'color': 0, 'x': -0.135634, 'y': 0.142330, 'z': 0.126765 }
ellipses['M2Gkey217.141106'] = { 'name': 'M2Gkey217.141106', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.329204, 'y': -0.031723, 'z': -0.079914 }
points['M2Gkey217.141106'] = { 'name': 'M2Gkey217.141106', 'color': 0, 'x': 0.329204, 'y': -0.031723, 'z': -0.079914 }
ellipses['M9Mkey217.141007'] = { 'name': 'M9Mkey217.141007', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.205622, 'y': -0.059965, 'z': -0.084318 }
points['M9Mkey217.141007'] = { 'name': 'M9Mkey217.141007', 'color': 0, 'x': -0.205622, 'y': -0.059965, 'z': -0.084318 }
ellipses['M2Space217.140997'] = { 'name': 'M2Space217.140997', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.323448, 'y': 0.035032, 'z': 0.007784 }
points['M2Space217.140997'] = { 'name': 'M2Space217.140997', 'color': 0, 'x': 0.323448, 'y': 0.035032, 'z': 0.007784 }
ellipses['M3Pinl217.141094'] = { 'name': 'M3Pinl217.141094', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.011027, 'y': 0.195586, 'z': -0.055742 }
points['M3Pinl217.141094'] = { 'name': 'M3Pinl217.141094', 'color': 0, 'x': 0.011027, 'y': 0.195586, 'z': -0.055742 }
ellipses['M2Rkey217.141050'] = { 'name': 'M2Rkey217.141050', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.257946, 'y': -0.067736, 'z': 0.098816 }
points['M2Rkey217.141050'] = { 'name': 'M2Rkey217.141050', 'color': 0, 'x': 0.257946, 'y': -0.067736, 'z': 0.098816 }
ellipses['M3Ckey217.141073'] = { 'name': 'M3Ckey217.141073', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.209652, 'y': 0.067054, 'z': 0.178165 }
points['M3Ckey217.141073'] = { 'name': 'M3Ckey217.141073', 'color': 0, 'x': -0.209652, 'y': 0.067054, 'z': 0.178165 }
ellipses['M2Kkey217.141041'] = { 'name': 'M2Kkey217.141041', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.259905, 'y': -0.014360, 'z': -0.035308 }
points['M2Kkey217.141041'] = { 'name': 'M2Kkey217.141041', 'color': 0, 'x': 0.259905, 'y': -0.014360, 'z': -0.035308 }
ellipses['M3Pkey217.141031'] = { 'name': 'M3Pkey217.141031', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.129473, 'y': 0.132884, 'z': 0.093249 }
points['M3Pkey217.141031'] = { 'name': 'M3Pkey217.141031', 'color': 0, 'x': -0.129473, 'y': 0.132884, 'z': 0.093249 }
ellipses['M3Hkey217.141093'] = { 'name': 'M3Hkey217.141093', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.148932, 'y': 0.123963, 'z': 0.092963 }
points['M3Hkey217.141093'] = { 'name': 'M3Hkey217.141093', 'color': 0, 'x': -0.148932, 'y': 0.123963, 'z': 0.092963 }
ellipses['M2Pinl217.141003'] = { 'name': 'M2Pinl217.141003', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.219671, 'y': -0.069647, 'z': 0.039763 }
points['M2Pinl217.141003'] = { 'name': 'M2Pinl217.141003', 'color': 0, 'x': 0.219671, 'y': -0.069647, 'z': 0.039763 }
ellipses['M2Midl217.141009'] = { 'name': 'M2Midl217.141009', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.263126, 'y': -0.111505, 'z': 0.126196 }
points['M2Midl217.141009'] = { 'name': 'M2Midl217.141009', 'color': 0, 'x': 0.263126, 'y': -0.111505, 'z': 0.126196 }
ellipses['M9Indr217.140998'] = { 'name': 'M9Indr217.140998', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.164693, 'y': -0.138370, 'z': -0.245098 }
points['M9Indr217.140998'] = { 'name': 'M9Indr217.140998', 'color': 0, 'x': -0.164693, 'y': -0.138370, 'z': -0.245098 }
ellipses['M9Space217.141079'] = { 'name': 'M9Space217.141079', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.235343, 'y': -0.149990, 'z': 0.059830 }
points['M9Space217.141079'] = { 'name': 'M9Space217.141079', 'color': 0, 'x': -0.235343, 'y': -0.149990, 'z': 0.059830 }
ellipses['M9Thml217.141021'] = { 'name': 'M9Thml217.141021', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.190025, 'y': -0.163757, 'z': -0.167469 }
points['M9Thml217.141021'] = { 'name': 'M9Thml217.141021', 'color': 0, 'x': -0.190025, 'y': -0.163757, 'z': -0.167469 }
ellipses['M3Bkey217.141059'] = { 'name': 'M3Bkey217.141059', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.171598, 'y': 0.077181, 'z': 0.029057 }
points['M3Bkey217.141059'] = { 'name': 'M3Bkey217.141059', 'color': 0, 'x': -0.171598, 'y': 0.077181, 'z': 0.029057 }
ellipses['M2Pkey217.141105'] = { 'name': 'M2Pkey217.141105', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.218861, 'y': 0.007596, 'z': -0.011023 }
points['M2Pkey217.141105'] = { 'name': 'M2Pkey217.141105', 'color': 0, 'x': 0.218861, 'y': 0.007596, 'z': -0.011023 }
ellipses['M3Rkey217.141025'] = { 'name': 'M3Rkey217.141025', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.109810, 'y': 0.115526, 'z': -0.071466 }
points['M3Rkey217.141025'] = { 'name': 'M3Rkey217.141025', 'color': 0, 'x': -0.109810, 'y': 0.115526, 'z': -0.071466 }
ellipses['M2Jkey217.141038'] = { 'name': 'M2Jkey217.141038', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.237717, 'y': 0.035006, 'z': -0.029776 }
points['M2Jkey217.141038'] = { 'name': 'M2Jkey217.141038', 'color': 0, 'x': 0.237717, 'y': 0.035006, 'z': -0.029776 }
ellipses['M3Wkey217.141104'] = { 'name': 'M3Wkey217.141104', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.149727, 'y': 0.163448, 'z': -0.011257 }
points['M3Wkey217.141104'] = { 'name': 'M3Wkey217.141104', 'color': 0, 'x': -0.149727, 'y': 0.163448, 'z': -0.011257 }
ellipses['M3Qkey217.141051'] = { 'name': 'M3Qkey217.141051', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.152728, 'y': 0.094969, 'z': -0.043425 }
points['M3Qkey217.141051'] = { 'name': 'M3Qkey217.141051', 'color': 0, 'x': -0.152728, 'y': 0.094969, 'z': -0.043425 }
ellipses['M3Lsft217.141088'] = { 'name': 'M3Lsft217.141088', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.100620, 'y': 0.154945, 'z': 0.087893 }
points['M3Lsft217.141088'] = { 'name': 'M3Lsft217.141088', 'color': 0, 'x': -0.100620, 'y': 0.154945, 'z': 0.087893 }
ellipses['M2Indr217.141062'] = { 'name': 'M2Indr217.141062', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.112450, 'y': 0.056015, 'z': -0.107768 }
points['M2Indr217.141062'] = { 'name': 'M2Indr217.141062', 'color': 0, 'x': 0.112450, 'y': 0.056015, 'z': -0.107768 }
ellipses['M9Okey217.141076'] = { 'name': 'M9Okey217.141076', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.171522, 'y': -0.137762, 'z': -0.028791 }
points['M9Okey217.141076'] = { 'name': 'M9Okey217.141076', 'color': 0, 'x': -0.171522, 'y': -0.137762, 'z': -0.028791 }
ellipses['M2Thmr217.141087'] = { 'name': 'M2Thmr217.141087', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.146467, 'y': 0.019115, 'z': -0.136171 }
points['M2Thmr217.141087'] = { 'name': 'M2Thmr217.141087', 'color': 0, 'x': 0.146467, 'y': 0.019115, 'z': -0.136171 }
ellipses['M2Ckey217.141092'] = { 'name': 'M2Ckey217.141092', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.401501, 'y': -0.104325, 'z': 0.110807 }
points['M2Ckey217.141092'] = { 'name': 'M2Ckey217.141092', 'color': 0, 'x': 0.401501, 'y': -0.104325, 'z': 0.110807 }
ellipses['M2Fkey217.141078'] = { 'name': 'M2Fkey217.141078', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.342477, 'y': 0.005425, 'z': 0.013923 }
points['M2Fkey217.141078'] = { 'name': 'M2Fkey217.141078', 'color': 0, 'x': 0.342477, 'y': 0.005425, 'z': 0.013923 }
ellipses['M3Jkey217.141017'] = { 'name': 'M3Jkey217.141017', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.198774, 'y': 0.010740, 'z': 0.029337 }
points['M3Jkey217.141017'] = { 'name': 'M3Jkey217.141017', 'color': 0, 'x': -0.198774, 'y': 0.010740, 'z': 0.029337 }
ellipses['M2Wkey217.141070'] = { 'name': 'M2Wkey217.141070', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.327558, 'y': -0.034000, 'z': 0.080405 }
points['M2Wkey217.141070'] = { 'name': 'M2Wkey217.141070', 'color': 0, 'x': 0.327558, 'y': -0.034000, 'z': 0.080405 }
ellipses['M2Zkey217.141015'] = { 'name': 'M2Zkey217.141015', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.315109, 'y': 0.002772, 'z': 0.045032 }
points['M2Zkey217.141015'] = { 'name': 'M2Zkey217.141015', 'color': 0, 'x': 0.315109, 'y': 0.002772, 'z': 0.045032 }
ellipses['M9Skey217.141004'] = { 'name': 'M9Skey217.141004', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.248854, 'y': -0.181241, 'z': 0.090787 }
points['M9Skey217.141004'] = { 'name': 'M9Skey217.141004', 'color': 0, 'x': -0.248854, 'y': -0.181241, 'z': 0.090787 }
ellipses['M9Midr217.141060'] = { 'name': 'M9Midr217.141060', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.124891, 'y': -0.064096, 'z': -0.255399 }
points['M9Midr217.141060'] = { 'name': 'M9Midr217.141060', 'color': 0, 'x': -0.124891, 'y': -0.064096, 'z': -0.255399 }
ellipses['M2Thml217.141005'] = { 'name': 'M2Thml217.141005', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.295957, 'y': -0.096901, 'z': 0.085413 }
points['M2Thml217.141005'] = { 'name': 'M2Thml217.141005', 'color': 0, 'x': 0.295957, 'y': -0.096901, 'z': 0.085413 }
ellipses['M2Nkey217.141052'] = { 'name': 'M2Nkey217.141052', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.163048, 'y': -0.042688, 'z': -0.017760 }
points['M2Nkey217.141052'] = { 'name': 'M2Nkey217.141052', 'color': 0, 'x': 0.163048, 'y': -0.042688, 'z': -0.017760 }
ellipses['M3Midl217.141103'] = { 'name': 'M3Midl217.141103', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.090798, 'y': 0.118432, 'z': -0.069921 }
points['M3Midl217.141103'] = { 'name': 'M3Midl217.141103', 'color': 0, 'x': -0.090798, 'y': 0.118432, 'z': -0.069921 }
ellipses['M2Ekey217.141011'] = { 'name': 'M2Ekey217.141011', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.360972, 'y': -0.097280, 'z': 0.154186 }
points['M2Ekey217.141011'] = { 'name': 'M2Ekey217.141011', 'color': 0, 'x': 0.360972, 'y': -0.097280, 'z': 0.154186 }
ellipses['M2Rinl217.141058'] = { 'name': 'M2Rinl217.141058', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.223006, 'y': -0.123498, 'z': 0.115151 }
points['M2Rinl217.141058'] = { 'name': 'M2Rinl217.141058', 'color': 0, 'x': 0.223006, 'y': -0.123498, 'z': 0.115151 }
ellipses['M3Rsft217.140995'] = { 'name': 'M3Rsft217.140995', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.136408, 'y': 0.138934, 'z': 0.003208 }
points['M3Rsft217.140995'] = { 'name': 'M3Rsft217.140995', 'color': 0, 'x': -0.136408, 'y': 0.138934, 'z': 0.003208 }
ellipses['M3Akey217.141006'] = { 'name': 'M3Akey217.141006', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.173638, 'y': 0.151371, 'z': 0.080631 }
points['M3Akey217.141006'] = { 'name': 'M3Akey217.141006', 'color': 0, 'x': -0.173638, 'y': 0.151371, 'z': 0.080631 }
ellipses['M3Pinr217.141057'] = { 'name': 'M3Pinr217.141057', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.110945, 'y': 0.226163, 'z': 0.023309 }
points['M3Pinr217.141057'] = { 'name': 'M3Pinr217.141057', 'color': 0, 'x': -0.110945, 'y': 0.226163, 'z': 0.023309 }
ellipses['M2Qkey217.141102'] = { 'name': 'M2Qkey217.141102', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.280871, 'y': 0.008500, 'z': -0.002471 }
points['M2Qkey217.141102'] = { 'name': 'M2Qkey217.141102', 'color': 0, 'x': 0.280871, 'y': 0.008500, 'z': -0.002471 }
ellipses['M9Vkey217.141018'] = { 'name': 'M9Vkey217.141018', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.145173, 'y': -0.050328, 'z': -0.053496 }
points['M9Vkey217.141018'] = { 'name': 'M9Vkey217.141018', 'color': 0, 'x': -0.145173, 'y': -0.050328, 'z': -0.053496 }
ellipses['M9Ckey217.141013'] = { 'name': 'M9Ckey217.141013', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.223245, 'y': -0.081354, 'z': -0.021430 }
points['M9Ckey217.141013'] = { 'name': 'M9Ckey217.141013', 'color': 0, 'x': -0.223245, 'y': -0.081354, 'z': -0.021430 }
ellipses['M3Thml217.141040'] = { 'name': 'M3Thml217.141040', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.192074, 'y': 0.042630, 'z': 0.003728 }
points['M3Thml217.141040'] = { 'name': 'M3Thml217.141040', 'color': 0, 'x': -0.192074, 'y': 0.042630, 'z': 0.003728 }
ellipses['M2Skey217.141089'] = { 'name': 'M2Skey217.141089', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': 0.286987, 'y': -0.003516, 'z': 0.037216 }
points['M2Skey217.141089'] = { 'name': 'M2Skey217.141089', 'color': 0, 'x': 0.286987, 'y': -0.003516, 'z': 0.037216 }
ellipses['M9Akey217.141086'] = { 'name': 'M9Akey217.141086', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.154149, 'y': -0.133306, 'z': 0.128911 }
points['M9Akey217.141086'] = { 'name': 'M9Akey217.141086', 'color': 0, 'x': -0.154149, 'y': -0.133306, 'z': 0.128911 }
ellipses['M3Lkey217.141097'] = { 'name': 'M3Lkey217.141097', 'color': 0, 'width': 0.000000, 'height': 0.000000, 'length': 0.000000 , 'x': -0.099959, 'y': 0.194166, 'z': 0.010595 }
points['M3Lkey217.141097'] = { 'name': 'M3Lkey217.141097', 'color': 0, 'x': -0.099959, 'y': 0.194166, 'z': 0.010595 }
var segments = 16, rings = 16, radius = -0.013323
var xaxislength = 0.666143
var yaxislength = 0.499622
var zaxislength = 0.498446
var min_x = -0.264642
var min_y = -0.251755
var min_z = -0.259952
var max_x = 0.401501
var max_y = 0.247867
var max_z = 0.238494
max = 0.461726
pc1 = 28
pc2 = 12
pc3 = 7
"""
if __name__ == "__main__":
    main()
